2023-09-02 07:30:12,467 INFO    StreamThr :9010 [internal.py:wandb_internal():86] W&B internal server running at pid: 9010, started at: 2023-09-02 07:30:12.466983
2023-09-02 07:30:12,470 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status
2023-09-02 07:30:12,471 INFO    WriterThread:9010 [datastore.py:open_for_write():85] open: /root/autodl-tmp/msc_ml/llm_thought/Train/SVAMP/wandb/run-20230902_073012-an06fkb2/run-an06fkb2.wandb
2023-09-02 07:30:12,471 DEBUG   SenderThread:9010 [sender.py:send():375] send: header
2023-09-02 07:30:13,657 DEBUG   SenderThread:9010 [sender.py:send():375] send: run
2023-09-02 07:30:14,323 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: check_version
2023-09-02 07:30:14,325 INFO    SenderThread:9010 [dir_watcher.py:__init__():219] watching files in: /root/autodl-tmp/msc_ml/llm_thought/Train/SVAMP/wandb/run-20230902_073012-an06fkb2/files
2023-09-02 07:30:14,326 INFO    SenderThread:9010 [sender.py:_start_run_threads():1124] run started: an06fkb2 with start time 1693611012.466384
2023-09-02 07:30:14,326 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: summary_record
2023-09-02 07:30:14,326 INFO    SenderThread:9010 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2023-09-02 07:30:14,326 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: check_version
2023-09-02 07:30:14,542 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: run_start
2023-09-02 07:30:14,551 DEBUG   HandlerThread:9010 [system_info.py:__init__():31] System info init
2023-09-02 07:30:14,551 DEBUG   HandlerThread:9010 [system_info.py:__init__():46] System info init done
2023-09-02 07:30:14,551 INFO    HandlerThread:9010 [system_monitor.py:start():181] Starting system monitor
2023-09-02 07:30:14,552 INFO    SystemMonitor:9010 [system_monitor.py:_start():145] Starting system asset monitoring threads
2023-09-02 07:30:14,552 INFO    HandlerThread:9010 [system_monitor.py:probe():201] Collecting system info
2023-09-02 07:30:14,553 INFO    SystemMonitor:9010 [interfaces.py:start():190] Started cpu monitoring
2023-09-02 07:30:14,553 INFO    SystemMonitor:9010 [interfaces.py:start():190] Started disk monitoring
2023-09-02 07:30:14,554 INFO    SystemMonitor:9010 [interfaces.py:start():190] Started gpu monitoring
2023-09-02 07:30:14,555 INFO    SystemMonitor:9010 [interfaces.py:start():190] Started memory monitoring
2023-09-02 07:30:14,555 INFO    SystemMonitor:9010 [interfaces.py:start():190] Started network monitoring
2023-09-02 07:30:14,574 DEBUG   HandlerThread:9010 [system_info.py:probe():195] Probing system
2023-09-02 07:30:14,581 DEBUG   HandlerThread:9010 [system_info.py:_probe_git():180] Probing git
2023-09-02 07:30:14,593 DEBUG   HandlerThread:9010 [system_info.py:_probe_git():188] Probing git done
2023-09-02 07:30:14,593 DEBUG   HandlerThread:9010 [system_info.py:probe():240] Probing system done
2023-09-02 07:30:14,593 DEBUG   HandlerThread:9010 [system_monitor.py:probe():210] {'os': 'Linux-5.4.0-99-generic-x86_64-with-glibc2.17', 'python': '3.8.10', 'heartbeatAt': '2023-09-01T23:30:14.574402', 'startedAt': '2023-09-01T23:30:12.459641', 'docker': None, 'cuda': None, 'args': (), 'state': 'running', 'program': '/root/autodl-tmp/msc_ml/llm_thought/Train/SVAMP/llama_2_trlx.py', 'codePath': 'Train/SVAMP/llama_2_trlx.py', 'git': {'remote': 'https://github.com/Greenoso/llm_thought.git', 'commit': '24305a3a985102cd18f50a6e8694b77e36025f3a'}, 'email': 'jiacheng.liu.19.ucl@gmail.com', 'root': '/root/autodl-tmp/msc_ml/llm_thought', 'host': 'autodl-container-55da118bfa-7e0e117f', 'username': 'root', 'executable': '/root/miniconda3/bin/python', 'cpu_count': 40, 'cpu_count_logical': 80, 'cpu_freq': {'current': 2999.864000000001, 'min': 0.0, 'max': 0.0}, 'cpu_freq_per_core': [{'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}, {'current': 2999.864, 'min': 0.0, 'max': 0.0}], 'disk': {'total': 30.0, 'used': 12.48031234741211}, 'gpu': 'Tesla V100S-PCIE-32GB', 'gpu_count': 2, 'gpu_devices': [{'name': 'Tesla V100S-PCIE-32GB', 'memory_total': 34359738368}, {'name': 'Tesla V100S-PCIE-32GB', 'memory_total': 34359738368}], 'memory': {'total': 629.8885612487793}}
2023-09-02 07:30:14,593 INFO    HandlerThread:9010 [system_monitor.py:probe():211] Finished collecting system info
2023-09-02 07:30:14,593 INFO    HandlerThread:9010 [system_monitor.py:probe():214] Publishing system info
2023-09-02 07:30:14,593 DEBUG   HandlerThread:9010 [system_info.py:_save_pip():51] Saving list of pip packages installed into the current environment
2023-09-02 07:30:14,594 DEBUG   HandlerThread:9010 [system_info.py:_save_pip():67] Saving pip packages done
2023-09-02 07:30:14,594 DEBUG   HandlerThread:9010 [system_info.py:_save_conda():74] Saving list of conda packages installed into the current environment
2023-09-02 07:30:15,327 INFO    Thread-12 :9010 [dir_watcher.py:_on_file_created():278] file/dir created: /root/autodl-tmp/msc_ml/llm_thought/Train/SVAMP/wandb/run-20230902_073012-an06fkb2/files/requirements.txt
2023-09-02 07:30:15,327 INFO    Thread-12 :9010 [dir_watcher.py:_on_file_created():278] file/dir created: /root/autodl-tmp/msc_ml/llm_thought/Train/SVAMP/wandb/run-20230902_073012-an06fkb2/files/wandb-summary.json
2023-09-02 07:30:15,327 INFO    Thread-12 :9010 [dir_watcher.py:_on_file_created():278] file/dir created: /root/autodl-tmp/msc_ml/llm_thought/Train/SVAMP/wandb/run-20230902_073012-an06fkb2/files/conda-environment.yaml
2023-09-02 07:30:17,838 DEBUG   HandlerThread:9010 [system_info.py:_save_conda():86] Saving conda packages done
2023-09-02 07:30:17,839 INFO    HandlerThread:9010 [system_monitor.py:probe():216] Finished publishing system info
2023-09-02 07:30:17,843 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:30:17,844 DEBUG   SenderThread:9010 [sender.py:send():375] send: files
2023-09-02 07:30:17,844 INFO    SenderThread:9010 [sender.py:_save_file():1378] saving file wandb-metadata.json with policy now
2023-09-02 07:30:17,851 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:30:17,851 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:30:18,233 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/ucl_2023_com00087/trlx/an06fkb2/wandb-metadata.json?Expires=1693697418&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IyJ78E4KYvK780tDzzhndQFjwFadOPGOOCxnnEhzBCNSc7E8DaWyD7Lp%2FjsL4oWwssBH%2FxAkxObYd9Ui3BBe1qGBk3FBeebmV%2BvnaQayanzemeyJJ%2BmoiQ%2BGrkoM37rmz05qOw%2BVSOV9tAi1btjqK0w%2BibuxM3fIy1LvGk7wKOt61kqykMvq15jBIZogsrTY4aqjRbNU2Ynxr0dFwLwNagX07gZIpvze5lDOvj0bxGCSIf9cUqZg4rHwuS5UCmugAbtjVuoHbcXtPoZscv4XGd0EvURQ6kDGAfSgZnLVvEf6ITTyxaRoY1V427sqctkNNzOP%2Fn9xlgSqxJFQrhtVuA%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/ucl_2023_com00087/trlx/an06fkb2/wandb-metadata.json?Expires=1693697418&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IyJ78E4KYvK780tDzzhndQFjwFadOPGOOCxnnEhzBCNSc7E8DaWyD7Lp%2FjsL4oWwssBH%2FxAkxObYd9Ui3BBe1qGBk3FBeebmV%2BvnaQayanzemeyJJ%2BmoiQ%2BGrkoM37rmz05qOw%2BVSOV9tAi1btjqK0w%2BibuxM3fIy1LvGk7wKOt61kqykMvq15jBIZogsrTY4aqjRbNU2Ynxr0dFwLwNagX07gZIpvze5lDOvj0bxGCSIf9cUqZg4rHwuS5UCmugAbtjVuoHbcXtPoZscv4XGd0EvURQ6kDGAfSgZnLVvEf6ITTyxaRoY1V427sqctkNNzOP%2Fn9xlgSqxJFQrhtVuA%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 07:30:18,233 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '9452'}
2023-09-02 07:30:18,233 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 07:30:18,241 DEBUG   SenderThread:9010 [sender.py:send():375] send: telemetry
2023-09-02 07:30:18,242 DEBUG   SenderThread:9010 [sender.py:send():375] send: config
2023-09-02 07:30:18,327 INFO    Thread-12 :9010 [dir_watcher.py:_on_file_modified():295] file/dir modified: /root/autodl-tmp/msc_ml/llm_thought/Train/SVAMP/wandb/run-20230902_073012-an06fkb2/files/conda-environment.yaml
2023-09-02 07:30:18,327 INFO    Thread-12 :9010 [dir_watcher.py:_on_file_created():278] file/dir created: /root/autodl-tmp/msc_ml/llm_thought/Train/SVAMP/wandb/run-20230902_073012-an06fkb2/files/wandb-metadata.json
2023-09-02 07:30:18,327 INFO    Thread-12 :9010 [dir_watcher.py:_on_file_created():278] file/dir created: /root/autodl-tmp/msc_ml/llm_thought/Train/SVAMP/wandb/run-20230902_073012-an06fkb2/files/output.log
2023-09-02 07:30:19,465 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/ucl_2023_com00087/trlx/an06fkb2/wandb-metadata.json?Expires=1693697418&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IyJ78E4KYvK780tDzzhndQFjwFadOPGOOCxnnEhzBCNSc7E8DaWyD7Lp%2FjsL4oWwssBH%2FxAkxObYd9Ui3BBe1qGBk3FBeebmV%2BvnaQayanzemeyJJ%2BmoiQ%2BGrkoM37rmz05qOw%2BVSOV9tAi1btjqK0w%2BibuxM3fIy1LvGk7wKOt61kqykMvq15jBIZogsrTY4aqjRbNU2Ynxr0dFwLwNagX07gZIpvze5lDOvj0bxGCSIf9cUqZg4rHwuS5UCmugAbtjVuoHbcXtPoZscv4XGd0EvURQ6kDGAfSgZnLVvEf6ITTyxaRoY1V427sqctkNNzOP%2Fn9xlgSqxJFQrhtVuA%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/ucl_2023_com00087/trlx/an06fkb2/wandb-metadata.json?Expires=1693697418&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IyJ78E4KYvK780tDzzhndQFjwFadOPGOOCxnnEhzBCNSc7E8DaWyD7Lp%2FjsL4oWwssBH%2FxAkxObYd9Ui3BBe1qGBk3FBeebmV%2BvnaQayanzemeyJJ%2BmoiQ%2BGrkoM37rmz05qOw%2BVSOV9tAi1btjqK0w%2BibuxM3fIy1LvGk7wKOt61kqykMvq15jBIZogsrTY4aqjRbNU2Ynxr0dFwLwNagX07gZIpvze5lDOvj0bxGCSIf9cUqZg4rHwuS5UCmugAbtjVuoHbcXtPoZscv4XGd0EvURQ6kDGAfSgZnLVvEf6ITTyxaRoY1V427sqctkNNzOP%2Fn9xlgSqxJFQrhtVuA%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 07:30:19,465 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '9452'}
2023-09-02 07:30:19,465 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 07:30:20,327 INFO    Thread-12 :9010 [dir_watcher.py:_on_file_modified():295] file/dir modified: /root/autodl-tmp/msc_ml/llm_thought/Train/SVAMP/wandb/run-20230902_073012-an06fkb2/files/output.log
2023-09-02 07:30:22,359 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/ucl_2023_com00087/trlx/an06fkb2/wandb-metadata.json?Expires=1693697418&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IyJ78E4KYvK780tDzzhndQFjwFadOPGOOCxnnEhzBCNSc7E8DaWyD7Lp%2FjsL4oWwssBH%2FxAkxObYd9Ui3BBe1qGBk3FBeebmV%2BvnaQayanzemeyJJ%2BmoiQ%2BGrkoM37rmz05qOw%2BVSOV9tAi1btjqK0w%2BibuxM3fIy1LvGk7wKOt61kqykMvq15jBIZogsrTY4aqjRbNU2Ynxr0dFwLwNagX07gZIpvze5lDOvj0bxGCSIf9cUqZg4rHwuS5UCmugAbtjVuoHbcXtPoZscv4XGd0EvURQ6kDGAfSgZnLVvEf6ITTyxaRoY1V427sqctkNNzOP%2Fn9xlgSqxJFQrhtVuA%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/ucl_2023_com00087/trlx/an06fkb2/wandb-metadata.json?Expires=1693697418&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IyJ78E4KYvK780tDzzhndQFjwFadOPGOOCxnnEhzBCNSc7E8DaWyD7Lp%2FjsL4oWwssBH%2FxAkxObYd9Ui3BBe1qGBk3FBeebmV%2BvnaQayanzemeyJJ%2BmoiQ%2BGrkoM37rmz05qOw%2BVSOV9tAi1btjqK0w%2BibuxM3fIy1LvGk7wKOt61kqykMvq15jBIZogsrTY4aqjRbNU2Ynxr0dFwLwNagX07gZIpvze5lDOvj0bxGCSIf9cUqZg4rHwuS5UCmugAbtjVuoHbcXtPoZscv4XGd0EvURQ6kDGAfSgZnLVvEf6ITTyxaRoY1V427sqctkNNzOP%2Fn9xlgSqxJFQrhtVuA%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 07:30:22,359 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '9452'}
2023-09-02 07:30:22,359 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 07:30:22,359 INFO    wandb-upload_0:9010 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/root/miniconda3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 711, in urlopen
    self._prepare_proxy(conn)
  File "/root/miniconda3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 1007, in _prepare_proxy
    conn.connect()
  File "/root/miniconda3/lib/python3.8/site-packages/urllib3/connection.py", line 419, in connect
    self.sock = ssl_wrap_socket(
  File "/root/miniconda3/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 449, in ssl_wrap_socket
    ssl_sock = _ssl_wrap_socket_impl(
  File "/root/miniconda3/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 493, in _ssl_wrap_socket_impl
    return ssl_context.wrap_socket(sock, server_hostname=server_hostname)
  File "/root/miniconda3/lib/python3.8/ssl.py", line 500, in wrap_socket
    return self.sslsocket_class._create(
  File "/root/miniconda3/lib/python3.8/ssl.py", line 1040, in _create
    self.do_handshake()
  File "/root/miniconda3/lib/python3.8/ssl.py", line 1309, in do_handshake
    self._sslobj.do_handshake()
ssl.SSLEOFError: EOF occurred in violation of protocol (_ssl.c:1131)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/root/miniconda3/lib/python3.8/site-packages/requests/adapters.py", line 486, in send
    resp = conn.urlopen(
  File "/root/miniconda3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 798, in urlopen
    retries = retries.increment(
  File "/root/miniconda3/lib/python3.8/site-packages/urllib3/util/retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/ucl_2023_com00087/trlx/an06fkb2/wandb-metadata.json?Expires=1693697418&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IyJ78E4KYvK780tDzzhndQFjwFadOPGOOCxnnEhzBCNSc7E8DaWyD7Lp%2FjsL4oWwssBH%2FxAkxObYd9Ui3BBe1qGBk3FBeebmV%2BvnaQayanzemeyJJ%2BmoiQ%2BGrkoM37rmz05qOw%2BVSOV9tAi1btjqK0w%2BibuxM3fIy1LvGk7wKOt61kqykMvq15jBIZogsrTY4aqjRbNU2Ynxr0dFwLwNagX07gZIpvze5lDOvj0bxGCSIf9cUqZg4rHwuS5UCmugAbtjVuoHbcXtPoZscv4XGd0EvURQ6kDGAfSgZnLVvEf6ITTyxaRoY1V427sqctkNNzOP%2Fn9xlgSqxJFQrhtVuA%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/root/miniconda3/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 2094, in upload_file
    response = self._upload_file_session.put(
  File "/root/miniconda3/lib/python3.8/site-packages/requests/sessions.py", line 649, in put
    return self.request("PUT", url, data=data, **kwargs)
  File "/root/miniconda3/lib/python3.8/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "/root/miniconda3/lib/python3.8/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
  File "/root/miniconda3/lib/python3.8/site-packages/requests/adapters.py", line 517, in send
    raise SSLError(e, request=request)
requests.exceptions.SSLError: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/ucl_2023_com00087/trlx/an06fkb2/wandb-metadata.json?Expires=1693697418&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IyJ78E4KYvK780tDzzhndQFjwFadOPGOOCxnnEhzBCNSc7E8DaWyD7Lp%2FjsL4oWwssBH%2FxAkxObYd9Ui3BBe1qGBk3FBeebmV%2BvnaQayanzemeyJJ%2BmoiQ%2BGrkoM37rmz05qOw%2BVSOV9tAi1btjqK0w%2BibuxM3fIy1LvGk7wKOt61kqykMvq15jBIZogsrTY4aqjRbNU2Ynxr0dFwLwNagX07gZIpvze5lDOvj0bxGCSIf9cUqZg4rHwuS5UCmugAbtjVuoHbcXtPoZscv4XGd0EvURQ6kDGAfSgZnLVvEf6ITTyxaRoY1V427sqctkNNzOP%2Fn9xlgSqxJFQrhtVuA%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/root/miniconda3/lib/python3.8/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/root/miniconda3/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 2123, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/root/miniconda3/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 2094, in upload_file
    response = self._upload_file_session.put(
  File "/root/miniconda3/lib/python3.8/site-packages/requests/sessions.py", line 649, in put
    return self.request("PUT", url, data=data, **kwargs)
  File "/root/miniconda3/lib/python3.8/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "/root/miniconda3/lib/python3.8/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
  File "/root/miniconda3/lib/python3.8/site-packages/requests/adapters.py", line 517, in send
    raise SSLError(e, request=request)
wandb.sdk.lib.retry.TransientError: None
2023-09-02 07:30:23,244 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:30:27,166 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/ucl_2023_com00087/trlx/an06fkb2/wandb-metadata.json?Expires=1693697418&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IyJ78E4KYvK780tDzzhndQFjwFadOPGOOCxnnEhzBCNSc7E8DaWyD7Lp%2FjsL4oWwssBH%2FxAkxObYd9Ui3BBe1qGBk3FBeebmV%2BvnaQayanzemeyJJ%2BmoiQ%2BGrkoM37rmz05qOw%2BVSOV9tAi1btjqK0w%2BibuxM3fIy1LvGk7wKOt61kqykMvq15jBIZogsrTY4aqjRbNU2Ynxr0dFwLwNagX07gZIpvze5lDOvj0bxGCSIf9cUqZg4rHwuS5UCmugAbtjVuoHbcXtPoZscv4XGd0EvURQ6kDGAfSgZnLVvEf6ITTyxaRoY1V427sqctkNNzOP%2Fn9xlgSqxJFQrhtVuA%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/ucl_2023_com00087/trlx/an06fkb2/wandb-metadata.json?Expires=1693697418&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IyJ78E4KYvK780tDzzhndQFjwFadOPGOOCxnnEhzBCNSc7E8DaWyD7Lp%2FjsL4oWwssBH%2FxAkxObYd9Ui3BBe1qGBk3FBeebmV%2BvnaQayanzemeyJJ%2BmoiQ%2BGrkoM37rmz05qOw%2BVSOV9tAi1btjqK0w%2BibuxM3fIy1LvGk7wKOt61kqykMvq15jBIZogsrTY4aqjRbNU2Ynxr0dFwLwNagX07gZIpvze5lDOvj0bxGCSIf9cUqZg4rHwuS5UCmugAbtjVuoHbcXtPoZscv4XGd0EvURQ6kDGAfSgZnLVvEf6ITTyxaRoY1V427sqctkNNzOP%2Fn9xlgSqxJFQrhtVuA%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 07:30:27,166 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '9452'}
2023-09-02 07:30:27,166 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 07:30:28,245 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:30:30,331 INFO    Thread-12 :9010 [dir_watcher.py:_on_file_modified():295] file/dir modified: /root/autodl-tmp/msc_ml/llm_thought/Train/SVAMP/wandb/run-20230902_073012-an06fkb2/files/output.log
2023-09-02 07:30:31,331 INFO    Thread-12 :9010 [dir_watcher.py:_on_file_modified():295] file/dir modified: /root/autodl-tmp/msc_ml/llm_thought/Train/SVAMP/wandb/run-20230902_073012-an06fkb2/files/output.log
2023-09-02 07:30:32,851 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:30:32,852 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:30:34,147 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:30:36,199 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/ucl_2023_com00087/trlx/an06fkb2/wandb-metadata.json?Expires=1693697418&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IyJ78E4KYvK780tDzzhndQFjwFadOPGOOCxnnEhzBCNSc7E8DaWyD7Lp%2FjsL4oWwssBH%2FxAkxObYd9Ui3BBe1qGBk3FBeebmV%2BvnaQayanzemeyJJ%2BmoiQ%2BGrkoM37rmz05qOw%2BVSOV9tAi1btjqK0w%2BibuxM3fIy1LvGk7wKOt61kqykMvq15jBIZogsrTY4aqjRbNU2Ynxr0dFwLwNagX07gZIpvze5lDOvj0bxGCSIf9cUqZg4rHwuS5UCmugAbtjVuoHbcXtPoZscv4XGd0EvURQ6kDGAfSgZnLVvEf6ITTyxaRoY1V427sqctkNNzOP%2Fn9xlgSqxJFQrhtVuA%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/ucl_2023_com00087/trlx/an06fkb2/wandb-metadata.json?Expires=1693697418&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IyJ78E4KYvK780tDzzhndQFjwFadOPGOOCxnnEhzBCNSc7E8DaWyD7Lp%2FjsL4oWwssBH%2FxAkxObYd9Ui3BBe1qGBk3FBeebmV%2BvnaQayanzemeyJJ%2BmoiQ%2BGrkoM37rmz05qOw%2BVSOV9tAi1btjqK0w%2BibuxM3fIy1LvGk7wKOt61kqykMvq15jBIZogsrTY4aqjRbNU2Ynxr0dFwLwNagX07gZIpvze5lDOvj0bxGCSIf9cUqZg4rHwuS5UCmugAbtjVuoHbcXtPoZscv4XGd0EvURQ6kDGAfSgZnLVvEf6ITTyxaRoY1V427sqctkNNzOP%2Fn9xlgSqxJFQrhtVuA%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 07:30:36,199 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '9452'}
2023-09-02 07:30:36,199 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 07:30:39,149 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:30:44,155 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:30:45,336 INFO    Thread-12 :9010 [dir_watcher.py:_on_file_modified():295] file/dir modified: /root/autodl-tmp/msc_ml/llm_thought/Train/SVAMP/wandb/run-20230902_073012-an06fkb2/files/config.yaml
2023-09-02 07:30:47,852 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:30:47,852 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:30:50,179 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:30:53,781 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/ucl_2023_com00087/trlx/an06fkb2/wandb-metadata.json?Expires=1693697418&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IyJ78E4KYvK780tDzzhndQFjwFadOPGOOCxnnEhzBCNSc7E8DaWyD7Lp%2FjsL4oWwssBH%2FxAkxObYd9Ui3BBe1qGBk3FBeebmV%2BvnaQayanzemeyJJ%2BmoiQ%2BGrkoM37rmz05qOw%2BVSOV9tAi1btjqK0w%2BibuxM3fIy1LvGk7wKOt61kqykMvq15jBIZogsrTY4aqjRbNU2Ynxr0dFwLwNagX07gZIpvze5lDOvj0bxGCSIf9cUqZg4rHwuS5UCmugAbtjVuoHbcXtPoZscv4XGd0EvURQ6kDGAfSgZnLVvEf6ITTyxaRoY1V427sqctkNNzOP%2Fn9xlgSqxJFQrhtVuA%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/ucl_2023_com00087/trlx/an06fkb2/wandb-metadata.json?Expires=1693697418&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IyJ78E4KYvK780tDzzhndQFjwFadOPGOOCxnnEhzBCNSc7E8DaWyD7Lp%2FjsL4oWwssBH%2FxAkxObYd9Ui3BBe1qGBk3FBeebmV%2BvnaQayanzemeyJJ%2BmoiQ%2BGrkoM37rmz05qOw%2BVSOV9tAi1btjqK0w%2BibuxM3fIy1LvGk7wKOt61kqykMvq15jBIZogsrTY4aqjRbNU2Ynxr0dFwLwNagX07gZIpvze5lDOvj0bxGCSIf9cUqZg4rHwuS5UCmugAbtjVuoHbcXtPoZscv4XGd0EvURQ6kDGAfSgZnLVvEf6ITTyxaRoY1V427sqctkNNzOP%2Fn9xlgSqxJFQrhtVuA%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 07:30:53,782 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '9452'}
2023-09-02 07:30:53,782 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 07:30:55,180 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:31:00,182 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:31:02,851 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:31:02,852 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:31:06,181 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:31:11,182 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:31:14,556 DEBUG   SystemMonitor:9010 [system_monitor.py:_start():159] Starting system metrics aggregation loop
2023-09-02 07:31:14,558 DEBUG   SenderThread:9010 [sender.py:send():375] send: stats
2023-09-02 07:31:16,559 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:31:17,852 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:31:17,852 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:31:22,326 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:31:22,347 INFO    Thread-12 :9010 [dir_watcher.py:_on_file_modified():295] file/dir modified: /root/autodl-tmp/msc_ml/llm_thought/Train/SVAMP/wandb/run-20230902_073012-an06fkb2/files/output.log
2023-09-02 07:31:24,837 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: partial_history
2023-09-02 07:31:27,349 INFO    Thread-12 :9010 [dir_watcher.py:_on_file_modified():295] file/dir modified: /root/autodl-tmp/msc_ml/llm_thought/Train/SVAMP/wandb/run-20230902_073012-an06fkb2/files/output.log
2023-09-02 07:31:27,842 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:31:29,752 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/ucl_2023_com00087/trlx/an06fkb2/wandb-metadata.json?Expires=1693697418&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IyJ78E4KYvK780tDzzhndQFjwFadOPGOOCxnnEhzBCNSc7E8DaWyD7Lp%2FjsL4oWwssBH%2FxAkxObYd9Ui3BBe1qGBk3FBeebmV%2BvnaQayanzemeyJJ%2BmoiQ%2BGrkoM37rmz05qOw%2BVSOV9tAi1btjqK0w%2BibuxM3fIy1LvGk7wKOt61kqykMvq15jBIZogsrTY4aqjRbNU2Ynxr0dFwLwNagX07gZIpvze5lDOvj0bxGCSIf9cUqZg4rHwuS5UCmugAbtjVuoHbcXtPoZscv4XGd0EvURQ6kDGAfSgZnLVvEf6ITTyxaRoY1V427sqctkNNzOP%2Fn9xlgSqxJFQrhtVuA%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/ucl_2023_com00087/trlx/an06fkb2/wandb-metadata.json?Expires=1693697418&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IyJ78E4KYvK780tDzzhndQFjwFadOPGOOCxnnEhzBCNSc7E8DaWyD7Lp%2FjsL4oWwssBH%2FxAkxObYd9Ui3BBe1qGBk3FBeebmV%2BvnaQayanzemeyJJ%2BmoiQ%2BGrkoM37rmz05qOw%2BVSOV9tAi1btjqK0w%2BibuxM3fIy1LvGk7wKOt61kqykMvq15jBIZogsrTY4aqjRbNU2Ynxr0dFwLwNagX07gZIpvze5lDOvj0bxGCSIf9cUqZg4rHwuS5UCmugAbtjVuoHbcXtPoZscv4XGd0EvURQ6kDGAfSgZnLVvEf6ITTyxaRoY1V427sqctkNNzOP%2Fn9xlgSqxJFQrhtVuA%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 07:31:29,753 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '9452'}
2023-09-02 07:31:29,753 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 07:31:32,843 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:31:32,852 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:31:32,852 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:31:38,124 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:31:43,125 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:31:44,559 DEBUG   SenderThread:9010 [sender.py:send():375] send: stats
2023-09-02 07:31:47,852 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:31:47,853 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:31:48,150 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:31:53,151 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:31:58,152 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:32:02,852 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:32:02,853 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:32:03,248 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:32:08,249 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:32:13,250 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:32:14,560 DEBUG   SenderThread:9010 [sender.py:send():375] send: stats
2023-09-02 07:32:15,365 INFO    Thread-12 :9010 [dir_watcher.py:_on_file_modified():295] file/dir modified: /root/autodl-tmp/msc_ml/llm_thought/Train/SVAMP/wandb/run-20230902_073012-an06fkb2/files/output.log
2023-09-02 07:32:17,852 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:32:17,853 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:32:19,143 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:32:24,145 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:32:29,146 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:32:32,853 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:32:32,853 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:32:35,122 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:32:40,123 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:32:40,202 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/ucl_2023_com00087/trlx/an06fkb2/wandb-metadata.json?Expires=1693697418&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IyJ78E4KYvK780tDzzhndQFjwFadOPGOOCxnnEhzBCNSc7E8DaWyD7Lp%2FjsL4oWwssBH%2FxAkxObYd9Ui3BBe1qGBk3FBeebmV%2BvnaQayanzemeyJJ%2BmoiQ%2BGrkoM37rmz05qOw%2BVSOV9tAi1btjqK0w%2BibuxM3fIy1LvGk7wKOt61kqykMvq15jBIZogsrTY4aqjRbNU2Ynxr0dFwLwNagX07gZIpvze5lDOvj0bxGCSIf9cUqZg4rHwuS5UCmugAbtjVuoHbcXtPoZscv4XGd0EvURQ6kDGAfSgZnLVvEf6ITTyxaRoY1V427sqctkNNzOP%2Fn9xlgSqxJFQrhtVuA%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/ucl_2023_com00087/trlx/an06fkb2/wandb-metadata.json?Expires=1693697418&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IyJ78E4KYvK780tDzzhndQFjwFadOPGOOCxnnEhzBCNSc7E8DaWyD7Lp%2FjsL4oWwssBH%2FxAkxObYd9Ui3BBe1qGBk3FBeebmV%2BvnaQayanzemeyJJ%2BmoiQ%2BGrkoM37rmz05qOw%2BVSOV9tAi1btjqK0w%2BibuxM3fIy1LvGk7wKOt61kqykMvq15jBIZogsrTY4aqjRbNU2Ynxr0dFwLwNagX07gZIpvze5lDOvj0bxGCSIf9cUqZg4rHwuS5UCmugAbtjVuoHbcXtPoZscv4XGd0EvURQ6kDGAfSgZnLVvEf6ITTyxaRoY1V427sqctkNNzOP%2Fn9xlgSqxJFQrhtVuA%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 07:32:40,202 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '9452'}
2023-09-02 07:32:40,202 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 07:32:44,561 DEBUG   SenderThread:9010 [sender.py:send():375] send: stats
2023-09-02 07:32:45,562 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:32:47,853 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:32:47,853 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:32:51,203 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:32:56,204 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:33:01,205 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:33:02,854 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:33:02,854 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:33:05,382 INFO    Thread-12 :9010 [dir_watcher.py:_on_file_modified():295] file/dir modified: /root/autodl-tmp/msc_ml/llm_thought/Train/SVAMP/wandb/run-20230902_073012-an06fkb2/files/output.log
2023-09-02 07:33:06,836 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:33:11,837 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:33:14,562 DEBUG   SenderThread:9010 [sender.py:send():375] send: stats
2023-09-02 07:33:17,563 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:33:17,853 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:33:17,854 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:33:23,180 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:33:28,182 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:33:32,854 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:33:32,854 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:33:34,113 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:33:39,114 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:33:44,115 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:33:44,563 DEBUG   SenderThread:9010 [sender.py:send():375] send: stats
2023-09-02 07:33:47,854 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:33:47,854 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:33:49,162 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:33:54,163 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:33:55,398 INFO    Thread-12 :9010 [dir_watcher.py:_on_file_modified():295] file/dir modified: /root/autodl-tmp/msc_ml/llm_thought/Train/SVAMP/wandb/run-20230902_073012-an06fkb2/files/output.log
2023-09-02 07:33:59,533 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:34:02,854 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:34:02,854 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:34:05,150 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:34:10,151 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:34:14,564 DEBUG   SenderThread:9010 [sender.py:send():375] send: stats
2023-09-02 07:34:15,565 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:34:17,854 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:34:17,855 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:34:21,177 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:34:26,179 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:34:31,180 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:34:32,855 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:34:32,855 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:34:36,180 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:34:41,181 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:34:44,565 DEBUG   SenderThread:9010 [sender.py:send():375] send: stats
2023-09-02 07:34:45,415 INFO    Thread-12 :9010 [dir_watcher.py:_on_file_modified():295] file/dir modified: /root/autodl-tmp/msc_ml/llm_thought/Train/SVAMP/wandb/run-20230902_073012-an06fkb2/files/output.log
2023-09-02 07:34:46,566 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:34:47,854 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:34:47,855 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:34:52,166 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:34:57,167 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:35:02,169 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:35:02,855 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:35:02,855 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:35:07,194 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:35:12,195 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:35:13,991 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/ucl_2023_com00087/trlx/an06fkb2/wandb-metadata.json?Expires=1693697418&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IyJ78E4KYvK780tDzzhndQFjwFadOPGOOCxnnEhzBCNSc7E8DaWyD7Lp%2FjsL4oWwssBH%2FxAkxObYd9Ui3BBe1qGBk3FBeebmV%2BvnaQayanzemeyJJ%2BmoiQ%2BGrkoM37rmz05qOw%2BVSOV9tAi1btjqK0w%2BibuxM3fIy1LvGk7wKOt61kqykMvq15jBIZogsrTY4aqjRbNU2Ynxr0dFwLwNagX07gZIpvze5lDOvj0bxGCSIf9cUqZg4rHwuS5UCmugAbtjVuoHbcXtPoZscv4XGd0EvURQ6kDGAfSgZnLVvEf6ITTyxaRoY1V427sqctkNNzOP%2Fn9xlgSqxJFQrhtVuA%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/ucl_2023_com00087/trlx/an06fkb2/wandb-metadata.json?Expires=1693697418&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IyJ78E4KYvK780tDzzhndQFjwFadOPGOOCxnnEhzBCNSc7E8DaWyD7Lp%2FjsL4oWwssBH%2FxAkxObYd9Ui3BBe1qGBk3FBeebmV%2BvnaQayanzemeyJJ%2BmoiQ%2BGrkoM37rmz05qOw%2BVSOV9tAi1btjqK0w%2BibuxM3fIy1LvGk7wKOt61kqykMvq15jBIZogsrTY4aqjRbNU2Ynxr0dFwLwNagX07gZIpvze5lDOvj0bxGCSIf9cUqZg4rHwuS5UCmugAbtjVuoHbcXtPoZscv4XGd0EvURQ6kDGAfSgZnLVvEf6ITTyxaRoY1V427sqctkNNzOP%2Fn9xlgSqxJFQrhtVuA%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 07:35:13,992 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '9452'}
2023-09-02 07:35:13,992 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 07:35:14,566 DEBUG   SenderThread:9010 [sender.py:send():375] send: stats
2023-09-02 07:35:17,568 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:35:17,857 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:35:17,858 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:35:21,432 INFO    Thread-12 :9010 [dir_watcher.py:_on_file_modified():295] file/dir modified: /root/autodl-tmp/msc_ml/llm_thought/Train/SVAMP/wandb/run-20230902_073012-an06fkb2/files/output.log
2023-09-02 07:35:22,783 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:35:27,784 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:35:32,785 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:35:32,857 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:35:32,857 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:35:38,179 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:35:43,180 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:35:44,567 DEBUG   SenderThread:9010 [sender.py:send():375] send: stats
2023-09-02 07:35:47,857 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:35:47,857 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:35:48,184 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:35:53,185 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:35:58,444 INFO    Thread-12 :9010 [dir_watcher.py:_on_file_modified():295] file/dir modified: /root/autodl-tmp/msc_ml/llm_thought/Train/SVAMP/wandb/run-20230902_073012-an06fkb2/files/output.log
2023-09-02 07:35:58,731 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:36:02,857 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:36:02,858 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:36:04,204 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:36:09,205 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:36:14,206 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:36:14,573 DEBUG   SenderThread:9010 [sender.py:send():375] send: stats
2023-09-02 07:36:17,857 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:36:17,858 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:36:20,166 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:36:25,167 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:36:30,169 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:36:32,858 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:36:32,858 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:36:36,121 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:36:41,122 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:36:44,570 DEBUG   SenderThread:9010 [sender.py:send():375] send: stats
2023-09-02 07:36:46,320 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:36:47,858 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:36:47,859 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:36:48,460 INFO    Thread-12 :9010 [dir_watcher.py:_on_file_modified():295] file/dir modified: /root/autodl-tmp/msc_ml/llm_thought/Train/SVAMP/wandb/run-20230902_073012-an06fkb2/files/output.log
2023-09-02 07:36:52,168 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:36:57,169 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:37:02,170 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:37:02,858 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:37:02,859 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:37:08,169 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:37:13,170 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:37:14,571 DEBUG   SenderThread:9010 [sender.py:send():375] send: stats
2023-09-02 07:37:17,858 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:37:17,859 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:37:19,110 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:37:24,111 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:37:29,112 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:37:32,858 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:37:32,859 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:37:34,185 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:37:38,478 INFO    Thread-12 :9010 [dir_watcher.py:_on_file_modified():295] file/dir modified: /root/autodl-tmp/msc_ml/llm_thought/Train/SVAMP/wandb/run-20230902_073012-an06fkb2/files/output.log
2023-09-02 07:37:39,930 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:37:44,573 DEBUG   SenderThread:9010 [sender.py:send():375] send: stats
2023-09-02 07:37:45,574 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:37:47,859 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:37:47,859 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:37:51,179 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:37:56,180 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:38:01,182 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:38:02,859 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:38:02,859 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:38:07,175 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:38:12,176 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:38:14,574 DEBUG   SenderThread:9010 [sender.py:send():375] send: stats
2023-09-02 07:38:17,575 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:38:17,859 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:38:17,860 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:38:22,711 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:38:24,494 INFO    Thread-12 :9010 [dir_watcher.py:_on_file_modified():295] file/dir modified: /root/autodl-tmp/msc_ml/llm_thought/Train/SVAMP/wandb/run-20230902_073012-an06fkb2/files/output.log
2023-09-02 07:38:27,713 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:38:32,714 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:38:32,860 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:38:32,861 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:38:38,137 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:38:43,138 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:38:44,575 DEBUG   SenderThread:9010 [sender.py:send():375] send: stats
2023-09-02 07:38:47,859 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:38:47,860 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:38:48,174 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:38:53,175 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:38:58,176 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:39:02,860 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:39:02,861 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:39:04,134 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:39:09,135 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:39:14,511 INFO    Thread-12 :9010 [dir_watcher.py:_on_file_modified():295] file/dir modified: /root/autodl-tmp/msc_ml/llm_thought/Train/SVAMP/wandb/run-20230902_073012-an06fkb2/files/output.log
2023-09-02 07:39:14,576 DEBUG   SenderThread:9010 [sender.py:send():375] send: stats
2023-09-02 07:39:14,578 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:39:17,860 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:39:17,860 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:39:20,147 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:39:25,148 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:39:30,149 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:39:32,860 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:39:32,861 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:39:36,139 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:39:41,140 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:39:44,577 DEBUG   SenderThread:9010 [sender.py:send():375] send: stats
2023-09-02 07:39:46,578 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:39:47,860 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:39:47,861 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:39:52,150 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:39:53,638 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/ucl_2023_com00087/trlx/an06fkb2/wandb-metadata.json?Expires=1693697418&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IyJ78E4KYvK780tDzzhndQFjwFadOPGOOCxnnEhzBCNSc7E8DaWyD7Lp%2FjsL4oWwssBH%2FxAkxObYd9Ui3BBe1qGBk3FBeebmV%2BvnaQayanzemeyJJ%2BmoiQ%2BGrkoM37rmz05qOw%2BVSOV9tAi1btjqK0w%2BibuxM3fIy1LvGk7wKOt61kqykMvq15jBIZogsrTY4aqjRbNU2Ynxr0dFwLwNagX07gZIpvze5lDOvj0bxGCSIf9cUqZg4rHwuS5UCmugAbtjVuoHbcXtPoZscv4XGd0EvURQ6kDGAfSgZnLVvEf6ITTyxaRoY1V427sqctkNNzOP%2Fn9xlgSqxJFQrhtVuA%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/ucl_2023_com00087/trlx/an06fkb2/wandb-metadata.json?Expires=1693697418&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IyJ78E4KYvK780tDzzhndQFjwFadOPGOOCxnnEhzBCNSc7E8DaWyD7Lp%2FjsL4oWwssBH%2FxAkxObYd9Ui3BBe1qGBk3FBeebmV%2BvnaQayanzemeyJJ%2BmoiQ%2BGrkoM37rmz05qOw%2BVSOV9tAi1btjqK0w%2BibuxM3fIy1LvGk7wKOt61kqykMvq15jBIZogsrTY4aqjRbNU2Ynxr0dFwLwNagX07gZIpvze5lDOvj0bxGCSIf9cUqZg4rHwuS5UCmugAbtjVuoHbcXtPoZscv4XGd0EvURQ6kDGAfSgZnLVvEf6ITTyxaRoY1V427sqctkNNzOP%2Fn9xlgSqxJFQrhtVuA%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 07:39:53,638 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '9452'}
2023-09-02 07:39:53,638 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 07:39:57,151 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:40:02,152 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:40:02,860 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:40:02,861 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:40:04,528 INFO    Thread-12 :9010 [dir_watcher.py:_on_file_modified():295] file/dir modified: /root/autodl-tmp/msc_ml/llm_thought/Train/SVAMP/wandb/run-20230902_073012-an06fkb2/files/output.log
2023-09-02 07:40:07,184 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:40:12,185 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:40:14,578 DEBUG   SenderThread:9010 [sender.py:send():375] send: stats
2023-09-02 07:40:17,580 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:40:17,861 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:40:17,862 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:40:23,126 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:40:28,127 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:40:32,861 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:40:32,862 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:40:34,117 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:40:39,118 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:40:44,119 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:40:44,580 DEBUG   SenderThread:9010 [sender.py:send():375] send: stats
2023-09-02 07:40:47,861 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:40:47,862 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:40:49,561 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:40:50,544 INFO    Thread-12 :9010 [dir_watcher.py:_on_file_modified():295] file/dir modified: /root/autodl-tmp/msc_ml/llm_thought/Train/SVAMP/wandb/run-20230902_073012-an06fkb2/files/output.log
2023-09-02 07:40:54,562 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:40:59,563 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:41:02,861 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:41:02,862 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:41:05,167 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:41:10,168 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:41:14,581 DEBUG   SenderThread:9010 [sender.py:send():375] send: stats
2023-09-02 07:41:15,582 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:41:17,861 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: stop_status
2023-09-02 07:41:17,862 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: stop_status
2023-09-02 07:41:21,130 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:41:21,312 DEBUG   SenderThread:9010 [sender.py:send():375] send: exit
2023-09-02 07:41:21,313 INFO    SenderThread:9010 [sender.py:send_exit():598] handling exit code: 1
2023-09-02 07:41:21,313 INFO    SenderThread:9010 [sender.py:send_exit():600] handling runtime: 666
2023-09-02 07:41:21,313 INFO    SenderThread:9010 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2023-09-02 07:41:21,314 INFO    SenderThread:9010 [sender.py:send_exit():606] send defer
2023-09-02 07:41:21,314 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: defer
2023-09-02 07:41:21,314 INFO    HandlerThread:9010 [handler.py:handle_request_defer():170] handle defer: 0
2023-09-02 07:41:21,314 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: defer
2023-09-02 07:41:21,314 INFO    SenderThread:9010 [sender.py:send_request_defer():622] handle sender defer: 0
2023-09-02 07:41:21,314 INFO    SenderThread:9010 [sender.py:transition_state():626] send defer: 1
2023-09-02 07:41:21,315 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: defer
2023-09-02 07:41:21,315 INFO    HandlerThread:9010 [handler.py:handle_request_defer():170] handle defer: 1
2023-09-02 07:41:21,315 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: defer
2023-09-02 07:41:21,315 INFO    SenderThread:9010 [sender.py:send_request_defer():622] handle sender defer: 1
2023-09-02 07:41:21,315 INFO    SenderThread:9010 [sender.py:transition_state():626] send defer: 2
2023-09-02 07:41:21,315 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: defer
2023-09-02 07:41:21,315 INFO    HandlerThread:9010 [handler.py:handle_request_defer():170] handle defer: 2
2023-09-02 07:41:21,315 INFO    HandlerThread:9010 [system_monitor.py:finish():190] Stopping system monitor
2023-09-02 07:41:21,316 INFO    HandlerThread:9010 [interfaces.py:finish():202] Joined cpu monitor
2023-09-02 07:41:21,316 DEBUG   SystemMonitor:9010 [system_monitor.py:_start():166] Finished system metrics aggregation loop
2023-09-02 07:41:21,316 DEBUG   SystemMonitor:9010 [system_monitor.py:_start():170] Publishing last batch of metrics
2023-09-02 07:41:21,318 INFO    HandlerThread:9010 [interfaces.py:finish():202] Joined disk monitor
2023-09-02 07:41:21,335 INFO    HandlerThread:9010 [interfaces.py:finish():202] Joined gpu monitor
2023-09-02 07:41:21,336 INFO    HandlerThread:9010 [interfaces.py:finish():202] Joined memory monitor
2023-09-02 07:41:21,336 INFO    HandlerThread:9010 [interfaces.py:finish():202] Joined network monitor
2023-09-02 07:41:21,336 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: defer
2023-09-02 07:41:21,336 INFO    SenderThread:9010 [sender.py:send_request_defer():622] handle sender defer: 2
2023-09-02 07:41:21,337 INFO    SenderThread:9010 [sender.py:transition_state():626] send defer: 3
2023-09-02 07:41:21,337 DEBUG   SenderThread:9010 [sender.py:send():375] send: stats
2023-09-02 07:41:21,337 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: defer
2023-09-02 07:41:21,337 INFO    HandlerThread:9010 [handler.py:handle_request_defer():170] handle defer: 3
2023-09-02 07:41:21,339 DEBUG   SenderThread:9010 [sender.py:send():375] send: history
2023-09-02 07:41:21,340 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: summary_record
2023-09-02 07:41:21,340 INFO    SenderThread:9010 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2023-09-02 07:41:21,340 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: defer
2023-09-02 07:41:21,340 INFO    SenderThread:9010 [sender.py:send_request_defer():622] handle sender defer: 3
2023-09-02 07:41:21,340 INFO    SenderThread:9010 [sender.py:transition_state():626] send defer: 4
2023-09-02 07:41:21,340 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: defer
2023-09-02 07:41:21,341 INFO    HandlerThread:9010 [handler.py:handle_request_defer():170] handle defer: 4
2023-09-02 07:41:21,341 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: defer
2023-09-02 07:41:21,341 INFO    SenderThread:9010 [sender.py:send_request_defer():622] handle sender defer: 4
2023-09-02 07:41:21,341 INFO    SenderThread:9010 [sender.py:transition_state():626] send defer: 5
2023-09-02 07:41:21,341 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: defer
2023-09-02 07:41:21,341 INFO    HandlerThread:9010 [handler.py:handle_request_defer():170] handle defer: 5
2023-09-02 07:41:21,342 DEBUG   SenderThread:9010 [sender.py:send():375] send: summary
2023-09-02 07:41:21,342 INFO    SenderThread:9010 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2023-09-02 07:41:21,342 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: defer
2023-09-02 07:41:21,342 INFO    SenderThread:9010 [sender.py:send_request_defer():622] handle sender defer: 5
2023-09-02 07:41:21,342 INFO    SenderThread:9010 [sender.py:transition_state():626] send defer: 6
2023-09-02 07:41:21,342 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: defer
2023-09-02 07:41:21,343 INFO    HandlerThread:9010 [handler.py:handle_request_defer():170] handle defer: 6
2023-09-02 07:41:21,343 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: defer
2023-09-02 07:41:21,343 INFO    SenderThread:9010 [sender.py:send_request_defer():622] handle sender defer: 6
2023-09-02 07:41:21,343 INFO    SenderThread:9010 [sender.py:transition_state():626] send defer: 7
2023-09-02 07:41:21,343 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: status_report
2023-09-02 07:41:21,343 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: defer
2023-09-02 07:41:21,343 INFO    HandlerThread:9010 [handler.py:handle_request_defer():170] handle defer: 7
2023-09-02 07:41:21,344 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: defer
2023-09-02 07:41:21,344 INFO    SenderThread:9010 [sender.py:send_request_defer():622] handle sender defer: 7
2023-09-02 07:41:21,558 INFO    Thread-12 :9010 [dir_watcher.py:_on_file_modified():295] file/dir modified: /root/autodl-tmp/msc_ml/llm_thought/Train/SVAMP/wandb/run-20230902_073012-an06fkb2/files/wandb-summary.json
2023-09-02 07:41:22,300 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: poll_exit
2023-09-02 07:41:22,559 INFO    Thread-12 :9010 [dir_watcher.py:_on_file_modified():295] file/dir modified: /root/autodl-tmp/msc_ml/llm_thought/Train/SVAMP/wandb/run-20230902_073012-an06fkb2/files/output.log
2023-09-02 07:41:23,559 INFO    Thread-12 :9010 [dir_watcher.py:_on_file_modified():295] file/dir modified: /root/autodl-tmp/msc_ml/llm_thought/Train/SVAMP/wandb/run-20230902_073012-an06fkb2/files/output.log
2023-09-02 07:41:23,860 INFO    SenderThread:9010 [sender.py:transition_state():626] send defer: 8
2023-09-02 07:41:23,860 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: poll_exit
2023-09-02 07:41:23,860 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: defer
2023-09-02 07:41:23,861 INFO    HandlerThread:9010 [handler.py:handle_request_defer():170] handle defer: 8
2023-09-02 07:41:23,861 DEBUG   SenderThread:9010 [sender.py:send_request():402] send_request: defer
2023-09-02 07:41:23,861 INFO    SenderThread:9010 [sender.py:send_request_defer():622] handle sender defer: 8
2023-09-02 07:41:23,869 INFO    SenderThread:9010 [sender.py:transition_state():626] send defer: 9
2023-09-02 07:41:23,869 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: defer
2023-09-02 07:41:23,869 INFO    HandlerThread:9010 [handler.py:handle_request_defer():170] handle defer: 9
2023-09-02 07:41:23,870 DEBUG   SenderThread:9010 [sender.py:send():375] send: artifact
2023-09-02 07:41:24,560 INFO    Thread-12 :9010 [dir_watcher.py:_on_file_modified():295] file/dir modified: /root/autodl-tmp/msc_ml/llm_thought/Train/SVAMP/wandb/run-20230902_073012-an06fkb2/files/output.log
2023-09-02 07:41:25,284 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/94466346/557270874/e90691de0f334b62cf2d8f7f82eceb55?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=iR4Tf%2B1vZGLNCdRl1e3cfjqlxI%2B1dcNzWEshrYfBKXnmVYhhEfWoPLsdoLxCFjhHdLM8cTsm%2BCxyGydq%2BEhbIokMyk7JxGY3Q35V3HBtvAZyo6xXWnM5VM2Kl5LvN%2FyGaglS3SA5RHsAAV%2FuZKuIV4715YnvvOgpxoP21AmHjLkTlcVGnalPJ6A4bFSuOHH1AoDt1XwophdYlI5pGTLuI7%2Fujz%2BhIOxhTX00CZtthObUmQ0QivqxKZrgarIQocEWVCm32%2FwUTeg1TD7QxKc%2FgV5TO7iXCpXtwpByOsHNByI3xCfuzLumSPkutWs6AOi0JxypzOvDXTMGlJpMWeqJNg%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/94466346/557270874/e90691de0f334b62cf2d8f7f82eceb55?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=iR4Tf%2B1vZGLNCdRl1e3cfjqlxI%2B1dcNzWEshrYfBKXnmVYhhEfWoPLsdoLxCFjhHdLM8cTsm%2BCxyGydq%2BEhbIokMyk7JxGY3Q35V3HBtvAZyo6xXWnM5VM2Kl5LvN%2FyGaglS3SA5RHsAAV%2FuZKuIV4715YnvvOgpxoP21AmHjLkTlcVGnalPJ6A4bFSuOHH1AoDt1XwophdYlI5pGTLuI7%2Fujz%2BhIOxhTX00CZtthObUmQ0QivqxKZrgarIQocEWVCm32%2FwUTeg1TD7QxKc%2FgV5TO7iXCpXtwpByOsHNByI3xCfuzLumSPkutWs6AOi0JxypzOvDXTMGlJpMWeqJNg%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 07:41:25,284 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': '6QaR3g8zS2LPLY9/guzrVQ==', 'Content-Type': 'application/json', 'Content-Length': '13956'}
2023-09-02 07:41:25,284 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 07:41:25,293 ERROR   wandb-upload_1:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/94466346/557270874/7cb72c6070c7b17c860e614316150974?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=dUD2Kxz1IQ8b6fM%2BN0AjL7Q%2B6uG0nfs4ITNHi%2Bl3oIMg0lyWyeFdrj%2FoAgvFVebnCKZHYwKm6oXsLGxf%2BnVxBNJcESbSeGqpb8g7yBg%2B5YuYeT80B5Ml5N4venmTnxBYvVCLauDN%2B%2B47WXgAQSkrlo77hmFYMhMOV7fjT4IJ7b9lk9638ONU%2FC%2FYc7cteZSiv0wSnFlcIgPeWj02eWkAQqTkec0BZlFthuQZsyYkdCGjBnem7cNxL9ZpzE%2BAfcSkaYh9vuOAYOKvnVhG5kzMHC%2BLnOqf2r4RcTZFCItqNkhgg%2B%2FD6qaixc6CcklFOpVloMpz2PKbnKYfakmp2W5cXQ%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/94466346/557270874/7cb72c6070c7b17c860e614316150974?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=dUD2Kxz1IQ8b6fM%2BN0AjL7Q%2B6uG0nfs4ITNHi%2Bl3oIMg0lyWyeFdrj%2FoAgvFVebnCKZHYwKm6oXsLGxf%2BnVxBNJcESbSeGqpb8g7yBg%2B5YuYeT80B5Ml5N4venmTnxBYvVCLauDN%2B%2B47WXgAQSkrlo77hmFYMhMOV7fjT4IJ7b9lk9638ONU%2FC%2FYc7cteZSiv0wSnFlcIgPeWj02eWkAQqTkec0BZlFthuQZsyYkdCGjBnem7cNxL9ZpzE%2BAfcSkaYh9vuOAYOKvnVhG5kzMHC%2BLnOqf2r4RcTZFCItqNkhgg%2B%2FD6qaixc6CcklFOpVloMpz2PKbnKYfakmp2W5cXQ%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 07:41:25,293 ERROR   wandb-upload_1:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': 'fLcsYHDHsXyGDmFDFhUJdA==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '4296'}
2023-09-02 07:41:25,293 ERROR   wandb-upload_1:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 07:41:26,481 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/94466346/557270874/e90691de0f334b62cf2d8f7f82eceb55?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=iR4Tf%2B1vZGLNCdRl1e3cfjqlxI%2B1dcNzWEshrYfBKXnmVYhhEfWoPLsdoLxCFjhHdLM8cTsm%2BCxyGydq%2BEhbIokMyk7JxGY3Q35V3HBtvAZyo6xXWnM5VM2Kl5LvN%2FyGaglS3SA5RHsAAV%2FuZKuIV4715YnvvOgpxoP21AmHjLkTlcVGnalPJ6A4bFSuOHH1AoDt1XwophdYlI5pGTLuI7%2Fujz%2BhIOxhTX00CZtthObUmQ0QivqxKZrgarIQocEWVCm32%2FwUTeg1TD7QxKc%2FgV5TO7iXCpXtwpByOsHNByI3xCfuzLumSPkutWs6AOi0JxypzOvDXTMGlJpMWeqJNg%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/94466346/557270874/e90691de0f334b62cf2d8f7f82eceb55?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=iR4Tf%2B1vZGLNCdRl1e3cfjqlxI%2B1dcNzWEshrYfBKXnmVYhhEfWoPLsdoLxCFjhHdLM8cTsm%2BCxyGydq%2BEhbIokMyk7JxGY3Q35V3HBtvAZyo6xXWnM5VM2Kl5LvN%2FyGaglS3SA5RHsAAV%2FuZKuIV4715YnvvOgpxoP21AmHjLkTlcVGnalPJ6A4bFSuOHH1AoDt1XwophdYlI5pGTLuI7%2Fujz%2BhIOxhTX00CZtthObUmQ0QivqxKZrgarIQocEWVCm32%2FwUTeg1TD7QxKc%2FgV5TO7iXCpXtwpByOsHNByI3xCfuzLumSPkutWs6AOi0JxypzOvDXTMGlJpMWeqJNg%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 07:41:26,481 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': '6QaR3g8zS2LPLY9/guzrVQ==', 'Content-Type': 'application/json', 'Content-Length': '13956'}
2023-09-02 07:41:26,481 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 07:41:26,596 ERROR   wandb-upload_1:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/94466346/557270874/7cb72c6070c7b17c860e614316150974?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=dUD2Kxz1IQ8b6fM%2BN0AjL7Q%2B6uG0nfs4ITNHi%2Bl3oIMg0lyWyeFdrj%2FoAgvFVebnCKZHYwKm6oXsLGxf%2BnVxBNJcESbSeGqpb8g7yBg%2B5YuYeT80B5Ml5N4venmTnxBYvVCLauDN%2B%2B47WXgAQSkrlo77hmFYMhMOV7fjT4IJ7b9lk9638ONU%2FC%2FYc7cteZSiv0wSnFlcIgPeWj02eWkAQqTkec0BZlFthuQZsyYkdCGjBnem7cNxL9ZpzE%2BAfcSkaYh9vuOAYOKvnVhG5kzMHC%2BLnOqf2r4RcTZFCItqNkhgg%2B%2FD6qaixc6CcklFOpVloMpz2PKbnKYfakmp2W5cXQ%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/94466346/557270874/7cb72c6070c7b17c860e614316150974?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=dUD2Kxz1IQ8b6fM%2BN0AjL7Q%2B6uG0nfs4ITNHi%2Bl3oIMg0lyWyeFdrj%2FoAgvFVebnCKZHYwKm6oXsLGxf%2BnVxBNJcESbSeGqpb8g7yBg%2B5YuYeT80B5Ml5N4venmTnxBYvVCLauDN%2B%2B47WXgAQSkrlo77hmFYMhMOV7fjT4IJ7b9lk9638ONU%2FC%2FYc7cteZSiv0wSnFlcIgPeWj02eWkAQqTkec0BZlFthuQZsyYkdCGjBnem7cNxL9ZpzE%2BAfcSkaYh9vuOAYOKvnVhG5kzMHC%2BLnOqf2r4RcTZFCItqNkhgg%2B%2FD6qaixc6CcklFOpVloMpz2PKbnKYfakmp2W5cXQ%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 07:41:26,596 ERROR   wandb-upload_1:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': 'fLcsYHDHsXyGDmFDFhUJdA==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '4296'}
2023-09-02 07:41:26,597 ERROR   wandb-upload_1:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 07:41:27,302 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:41:29,048 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/94466346/557270874/e90691de0f334b62cf2d8f7f82eceb55?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=iR4Tf%2B1vZGLNCdRl1e3cfjqlxI%2B1dcNzWEshrYfBKXnmVYhhEfWoPLsdoLxCFjhHdLM8cTsm%2BCxyGydq%2BEhbIokMyk7JxGY3Q35V3HBtvAZyo6xXWnM5VM2Kl5LvN%2FyGaglS3SA5RHsAAV%2FuZKuIV4715YnvvOgpxoP21AmHjLkTlcVGnalPJ6A4bFSuOHH1AoDt1XwophdYlI5pGTLuI7%2Fujz%2BhIOxhTX00CZtthObUmQ0QivqxKZrgarIQocEWVCm32%2FwUTeg1TD7QxKc%2FgV5TO7iXCpXtwpByOsHNByI3xCfuzLumSPkutWs6AOi0JxypzOvDXTMGlJpMWeqJNg%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/94466346/557270874/e90691de0f334b62cf2d8f7f82eceb55?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=iR4Tf%2B1vZGLNCdRl1e3cfjqlxI%2B1dcNzWEshrYfBKXnmVYhhEfWoPLsdoLxCFjhHdLM8cTsm%2BCxyGydq%2BEhbIokMyk7JxGY3Q35V3HBtvAZyo6xXWnM5VM2Kl5LvN%2FyGaglS3SA5RHsAAV%2FuZKuIV4715YnvvOgpxoP21AmHjLkTlcVGnalPJ6A4bFSuOHH1AoDt1XwophdYlI5pGTLuI7%2Fujz%2BhIOxhTX00CZtthObUmQ0QivqxKZrgarIQocEWVCm32%2FwUTeg1TD7QxKc%2FgV5TO7iXCpXtwpByOsHNByI3xCfuzLumSPkutWs6AOi0JxypzOvDXTMGlJpMWeqJNg%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 07:41:29,048 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': '6QaR3g8zS2LPLY9/guzrVQ==', 'Content-Type': 'application/json', 'Content-Length': '13956'}
2023-09-02 07:41:29,048 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 07:41:29,049 INFO    wandb-upload_2:9010 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/root/miniconda3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 711, in urlopen
    self._prepare_proxy(conn)
  File "/root/miniconda3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 1007, in _prepare_proxy
    conn.connect()
  File "/root/miniconda3/lib/python3.8/site-packages/urllib3/connection.py", line 419, in connect
    self.sock = ssl_wrap_socket(
  File "/root/miniconda3/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 449, in ssl_wrap_socket
    ssl_sock = _ssl_wrap_socket_impl(
  File "/root/miniconda3/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 493, in _ssl_wrap_socket_impl
    return ssl_context.wrap_socket(sock, server_hostname=server_hostname)
  File "/root/miniconda3/lib/python3.8/ssl.py", line 500, in wrap_socket
    return self.sslsocket_class._create(
  File "/root/miniconda3/lib/python3.8/ssl.py", line 1040, in _create
    self.do_handshake()
  File "/root/miniconda3/lib/python3.8/ssl.py", line 1309, in do_handshake
    self._sslobj.do_handshake()
ssl.SSLEOFError: EOF occurred in violation of protocol (_ssl.c:1131)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/root/miniconda3/lib/python3.8/site-packages/requests/adapters.py", line 486, in send
    resp = conn.urlopen(
  File "/root/miniconda3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 798, in urlopen
    retries = retries.increment(
  File "/root/miniconda3/lib/python3.8/site-packages/urllib3/util/retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/94466346/557270874/e90691de0f334b62cf2d8f7f82eceb55?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=iR4Tf%2B1vZGLNCdRl1e3cfjqlxI%2B1dcNzWEshrYfBKXnmVYhhEfWoPLsdoLxCFjhHdLM8cTsm%2BCxyGydq%2BEhbIokMyk7JxGY3Q35V3HBtvAZyo6xXWnM5VM2Kl5LvN%2FyGaglS3SA5RHsAAV%2FuZKuIV4715YnvvOgpxoP21AmHjLkTlcVGnalPJ6A4bFSuOHH1AoDt1XwophdYlI5pGTLuI7%2Fujz%2BhIOxhTX00CZtthObUmQ0QivqxKZrgarIQocEWVCm32%2FwUTeg1TD7QxKc%2FgV5TO7iXCpXtwpByOsHNByI3xCfuzLumSPkutWs6AOi0JxypzOvDXTMGlJpMWeqJNg%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/root/miniconda3/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 2094, in upload_file
    response = self._upload_file_session.put(
  File "/root/miniconda3/lib/python3.8/site-packages/requests/sessions.py", line 649, in put
    return self.request("PUT", url, data=data, **kwargs)
  File "/root/miniconda3/lib/python3.8/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "/root/miniconda3/lib/python3.8/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
  File "/root/miniconda3/lib/python3.8/site-packages/requests/adapters.py", line 517, in send
    raise SSLError(e, request=request)
requests.exceptions.SSLError: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/94466346/557270874/e90691de0f334b62cf2d8f7f82eceb55?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=iR4Tf%2B1vZGLNCdRl1e3cfjqlxI%2B1dcNzWEshrYfBKXnmVYhhEfWoPLsdoLxCFjhHdLM8cTsm%2BCxyGydq%2BEhbIokMyk7JxGY3Q35V3HBtvAZyo6xXWnM5VM2Kl5LvN%2FyGaglS3SA5RHsAAV%2FuZKuIV4715YnvvOgpxoP21AmHjLkTlcVGnalPJ6A4bFSuOHH1AoDt1XwophdYlI5pGTLuI7%2Fujz%2BhIOxhTX00CZtthObUmQ0QivqxKZrgarIQocEWVCm32%2FwUTeg1TD7QxKc%2FgV5TO7iXCpXtwpByOsHNByI3xCfuzLumSPkutWs6AOi0JxypzOvDXTMGlJpMWeqJNg%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/root/miniconda3/lib/python3.8/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/root/miniconda3/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 2123, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/root/miniconda3/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 2094, in upload_file
    response = self._upload_file_session.put(
  File "/root/miniconda3/lib/python3.8/site-packages/requests/sessions.py", line 649, in put
    return self.request("PUT", url, data=data, **kwargs)
  File "/root/miniconda3/lib/python3.8/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "/root/miniconda3/lib/python3.8/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
  File "/root/miniconda3/lib/python3.8/site-packages/requests/adapters.py", line 517, in send
    raise SSLError(e, request=request)
wandb.sdk.lib.retry.TransientError: None
2023-09-02 07:41:29,432 ERROR   wandb-upload_1:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/94466346/557270874/7cb72c6070c7b17c860e614316150974?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=dUD2Kxz1IQ8b6fM%2BN0AjL7Q%2B6uG0nfs4ITNHi%2Bl3oIMg0lyWyeFdrj%2FoAgvFVebnCKZHYwKm6oXsLGxf%2BnVxBNJcESbSeGqpb8g7yBg%2B5YuYeT80B5Ml5N4venmTnxBYvVCLauDN%2B%2B47WXgAQSkrlo77hmFYMhMOV7fjT4IJ7b9lk9638ONU%2FC%2FYc7cteZSiv0wSnFlcIgPeWj02eWkAQqTkec0BZlFthuQZsyYkdCGjBnem7cNxL9ZpzE%2BAfcSkaYh9vuOAYOKvnVhG5kzMHC%2BLnOqf2r4RcTZFCItqNkhgg%2B%2FD6qaixc6CcklFOpVloMpz2PKbnKYfakmp2W5cXQ%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/94466346/557270874/7cb72c6070c7b17c860e614316150974?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=dUD2Kxz1IQ8b6fM%2BN0AjL7Q%2B6uG0nfs4ITNHi%2Bl3oIMg0lyWyeFdrj%2FoAgvFVebnCKZHYwKm6oXsLGxf%2BnVxBNJcESbSeGqpb8g7yBg%2B5YuYeT80B5Ml5N4venmTnxBYvVCLauDN%2B%2B47WXgAQSkrlo77hmFYMhMOV7fjT4IJ7b9lk9638ONU%2FC%2FYc7cteZSiv0wSnFlcIgPeWj02eWkAQqTkec0BZlFthuQZsyYkdCGjBnem7cNxL9ZpzE%2BAfcSkaYh9vuOAYOKvnVhG5kzMHC%2BLnOqf2r4RcTZFCItqNkhgg%2B%2FD6qaixc6CcklFOpVloMpz2PKbnKYfakmp2W5cXQ%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 07:41:29,432 ERROR   wandb-upload_1:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': 'fLcsYHDHsXyGDmFDFhUJdA==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '4296'}
2023-09-02 07:41:29,432 ERROR   wandb-upload_1:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 07:41:29,432 INFO    wandb-upload_1:9010 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/root/miniconda3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 711, in urlopen
    self._prepare_proxy(conn)
  File "/root/miniconda3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 1007, in _prepare_proxy
    conn.connect()
  File "/root/miniconda3/lib/python3.8/site-packages/urllib3/connection.py", line 419, in connect
    self.sock = ssl_wrap_socket(
  File "/root/miniconda3/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 449, in ssl_wrap_socket
    ssl_sock = _ssl_wrap_socket_impl(
  File "/root/miniconda3/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 493, in _ssl_wrap_socket_impl
    return ssl_context.wrap_socket(sock, server_hostname=server_hostname)
  File "/root/miniconda3/lib/python3.8/ssl.py", line 500, in wrap_socket
    return self.sslsocket_class._create(
  File "/root/miniconda3/lib/python3.8/ssl.py", line 1040, in _create
    self.do_handshake()
  File "/root/miniconda3/lib/python3.8/ssl.py", line 1309, in do_handshake
    self._sslobj.do_handshake()
ssl.SSLEOFError: EOF occurred in violation of protocol (_ssl.c:1131)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/root/miniconda3/lib/python3.8/site-packages/requests/adapters.py", line 486, in send
    resp = conn.urlopen(
  File "/root/miniconda3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 798, in urlopen
    retries = retries.increment(
  File "/root/miniconda3/lib/python3.8/site-packages/urllib3/util/retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/94466346/557270874/7cb72c6070c7b17c860e614316150974?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=dUD2Kxz1IQ8b6fM%2BN0AjL7Q%2B6uG0nfs4ITNHi%2Bl3oIMg0lyWyeFdrj%2FoAgvFVebnCKZHYwKm6oXsLGxf%2BnVxBNJcESbSeGqpb8g7yBg%2B5YuYeT80B5Ml5N4venmTnxBYvVCLauDN%2B%2B47WXgAQSkrlo77hmFYMhMOV7fjT4IJ7b9lk9638ONU%2FC%2FYc7cteZSiv0wSnFlcIgPeWj02eWkAQqTkec0BZlFthuQZsyYkdCGjBnem7cNxL9ZpzE%2BAfcSkaYh9vuOAYOKvnVhG5kzMHC%2BLnOqf2r4RcTZFCItqNkhgg%2B%2FD6qaixc6CcklFOpVloMpz2PKbnKYfakmp2W5cXQ%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/root/miniconda3/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 2094, in upload_file
    response = self._upload_file_session.put(
  File "/root/miniconda3/lib/python3.8/site-packages/requests/sessions.py", line 649, in put
    return self.request("PUT", url, data=data, **kwargs)
  File "/root/miniconda3/lib/python3.8/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "/root/miniconda3/lib/python3.8/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
  File "/root/miniconda3/lib/python3.8/site-packages/requests/adapters.py", line 517, in send
    raise SSLError(e, request=request)
requests.exceptions.SSLError: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/94466346/557270874/7cb72c6070c7b17c860e614316150974?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=dUD2Kxz1IQ8b6fM%2BN0AjL7Q%2B6uG0nfs4ITNHi%2Bl3oIMg0lyWyeFdrj%2FoAgvFVebnCKZHYwKm6oXsLGxf%2BnVxBNJcESbSeGqpb8g7yBg%2B5YuYeT80B5Ml5N4venmTnxBYvVCLauDN%2B%2B47WXgAQSkrlo77hmFYMhMOV7fjT4IJ7b9lk9638ONU%2FC%2FYc7cteZSiv0wSnFlcIgPeWj02eWkAQqTkec0BZlFthuQZsyYkdCGjBnem7cNxL9ZpzE%2BAfcSkaYh9vuOAYOKvnVhG5kzMHC%2BLnOqf2r4RcTZFCItqNkhgg%2B%2FD6qaixc6CcklFOpVloMpz2PKbnKYfakmp2W5cXQ%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/root/miniconda3/lib/python3.8/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/root/miniconda3/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 2123, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/root/miniconda3/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 2094, in upload_file
    response = self._upload_file_session.put(
  File "/root/miniconda3/lib/python3.8/site-packages/requests/sessions.py", line 649, in put
    return self.request("PUT", url, data=data, **kwargs)
  File "/root/miniconda3/lib/python3.8/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "/root/miniconda3/lib/python3.8/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
  File "/root/miniconda3/lib/python3.8/site-packages/requests/adapters.py", line 517, in send
    raise SSLError(e, request=request)
wandb.sdk.lib.retry.TransientError: None
2023-09-02 07:41:32,304 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:41:34,068 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/94466346/557270874/e90691de0f334b62cf2d8f7f82eceb55?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=iR4Tf%2B1vZGLNCdRl1e3cfjqlxI%2B1dcNzWEshrYfBKXnmVYhhEfWoPLsdoLxCFjhHdLM8cTsm%2BCxyGydq%2BEhbIokMyk7JxGY3Q35V3HBtvAZyo6xXWnM5VM2Kl5LvN%2FyGaglS3SA5RHsAAV%2FuZKuIV4715YnvvOgpxoP21AmHjLkTlcVGnalPJ6A4bFSuOHH1AoDt1XwophdYlI5pGTLuI7%2Fujz%2BhIOxhTX00CZtthObUmQ0QivqxKZrgarIQocEWVCm32%2FwUTeg1TD7QxKc%2FgV5TO7iXCpXtwpByOsHNByI3xCfuzLumSPkutWs6AOi0JxypzOvDXTMGlJpMWeqJNg%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/94466346/557270874/e90691de0f334b62cf2d8f7f82eceb55?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=iR4Tf%2B1vZGLNCdRl1e3cfjqlxI%2B1dcNzWEshrYfBKXnmVYhhEfWoPLsdoLxCFjhHdLM8cTsm%2BCxyGydq%2BEhbIokMyk7JxGY3Q35V3HBtvAZyo6xXWnM5VM2Kl5LvN%2FyGaglS3SA5RHsAAV%2FuZKuIV4715YnvvOgpxoP21AmHjLkTlcVGnalPJ6A4bFSuOHH1AoDt1XwophdYlI5pGTLuI7%2Fujz%2BhIOxhTX00CZtthObUmQ0QivqxKZrgarIQocEWVCm32%2FwUTeg1TD7QxKc%2FgV5TO7iXCpXtwpByOsHNByI3xCfuzLumSPkutWs6AOi0JxypzOvDXTMGlJpMWeqJNg%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 07:41:34,068 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': '6QaR3g8zS2LPLY9/guzrVQ==', 'Content-Type': 'application/json', 'Content-Length': '13956'}
2023-09-02 07:41:34,068 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 07:41:34,417 ERROR   wandb-upload_1:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/94466346/557270874/7cb72c6070c7b17c860e614316150974?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=dUD2Kxz1IQ8b6fM%2BN0AjL7Q%2B6uG0nfs4ITNHi%2Bl3oIMg0lyWyeFdrj%2FoAgvFVebnCKZHYwKm6oXsLGxf%2BnVxBNJcESbSeGqpb8g7yBg%2B5YuYeT80B5Ml5N4venmTnxBYvVCLauDN%2B%2B47WXgAQSkrlo77hmFYMhMOV7fjT4IJ7b9lk9638ONU%2FC%2FYc7cteZSiv0wSnFlcIgPeWj02eWkAQqTkec0BZlFthuQZsyYkdCGjBnem7cNxL9ZpzE%2BAfcSkaYh9vuOAYOKvnVhG5kzMHC%2BLnOqf2r4RcTZFCItqNkhgg%2B%2FD6qaixc6CcklFOpVloMpz2PKbnKYfakmp2W5cXQ%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/94466346/557270874/7cb72c6070c7b17c860e614316150974?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=dUD2Kxz1IQ8b6fM%2BN0AjL7Q%2B6uG0nfs4ITNHi%2Bl3oIMg0lyWyeFdrj%2FoAgvFVebnCKZHYwKm6oXsLGxf%2BnVxBNJcESbSeGqpb8g7yBg%2B5YuYeT80B5Ml5N4venmTnxBYvVCLauDN%2B%2B47WXgAQSkrlo77hmFYMhMOV7fjT4IJ7b9lk9638ONU%2FC%2FYc7cteZSiv0wSnFlcIgPeWj02eWkAQqTkec0BZlFthuQZsyYkdCGjBnem7cNxL9ZpzE%2BAfcSkaYh9vuOAYOKvnVhG5kzMHC%2BLnOqf2r4RcTZFCItqNkhgg%2B%2FD6qaixc6CcklFOpVloMpz2PKbnKYfakmp2W5cXQ%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 07:41:34,417 ERROR   wandb-upload_1:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': 'fLcsYHDHsXyGDmFDFhUJdA==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '4296'}
2023-09-02 07:41:34,417 ERROR   wandb-upload_1:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 07:41:37,305 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:41:42,307 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:41:42,608 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/94466346/557270874/e90691de0f334b62cf2d8f7f82eceb55?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=iR4Tf%2B1vZGLNCdRl1e3cfjqlxI%2B1dcNzWEshrYfBKXnmVYhhEfWoPLsdoLxCFjhHdLM8cTsm%2BCxyGydq%2BEhbIokMyk7JxGY3Q35V3HBtvAZyo6xXWnM5VM2Kl5LvN%2FyGaglS3SA5RHsAAV%2FuZKuIV4715YnvvOgpxoP21AmHjLkTlcVGnalPJ6A4bFSuOHH1AoDt1XwophdYlI5pGTLuI7%2Fujz%2BhIOxhTX00CZtthObUmQ0QivqxKZrgarIQocEWVCm32%2FwUTeg1TD7QxKc%2FgV5TO7iXCpXtwpByOsHNByI3xCfuzLumSPkutWs6AOi0JxypzOvDXTMGlJpMWeqJNg%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/94466346/557270874/e90691de0f334b62cf2d8f7f82eceb55?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=iR4Tf%2B1vZGLNCdRl1e3cfjqlxI%2B1dcNzWEshrYfBKXnmVYhhEfWoPLsdoLxCFjhHdLM8cTsm%2BCxyGydq%2BEhbIokMyk7JxGY3Q35V3HBtvAZyo6xXWnM5VM2Kl5LvN%2FyGaglS3SA5RHsAAV%2FuZKuIV4715YnvvOgpxoP21AmHjLkTlcVGnalPJ6A4bFSuOHH1AoDt1XwophdYlI5pGTLuI7%2Fujz%2BhIOxhTX00CZtthObUmQ0QivqxKZrgarIQocEWVCm32%2FwUTeg1TD7QxKc%2FgV5TO7iXCpXtwpByOsHNByI3xCfuzLumSPkutWs6AOi0JxypzOvDXTMGlJpMWeqJNg%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 07:41:42,609 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': '6QaR3g8zS2LPLY9/guzrVQ==', 'Content-Type': 'application/json', 'Content-Length': '13956'}
2023-09-02 07:41:42,609 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 07:41:43,073 ERROR   wandb-upload_1:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/94466346/557270874/7cb72c6070c7b17c860e614316150974?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=dUD2Kxz1IQ8b6fM%2BN0AjL7Q%2B6uG0nfs4ITNHi%2Bl3oIMg0lyWyeFdrj%2FoAgvFVebnCKZHYwKm6oXsLGxf%2BnVxBNJcESbSeGqpb8g7yBg%2B5YuYeT80B5Ml5N4venmTnxBYvVCLauDN%2B%2B47WXgAQSkrlo77hmFYMhMOV7fjT4IJ7b9lk9638ONU%2FC%2FYc7cteZSiv0wSnFlcIgPeWj02eWkAQqTkec0BZlFthuQZsyYkdCGjBnem7cNxL9ZpzE%2BAfcSkaYh9vuOAYOKvnVhG5kzMHC%2BLnOqf2r4RcTZFCItqNkhgg%2B%2FD6qaixc6CcklFOpVloMpz2PKbnKYfakmp2W5cXQ%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/94466346/557270874/7cb72c6070c7b17c860e614316150974?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=dUD2Kxz1IQ8b6fM%2BN0AjL7Q%2B6uG0nfs4ITNHi%2Bl3oIMg0lyWyeFdrj%2FoAgvFVebnCKZHYwKm6oXsLGxf%2BnVxBNJcESbSeGqpb8g7yBg%2B5YuYeT80B5Ml5N4venmTnxBYvVCLauDN%2B%2B47WXgAQSkrlo77hmFYMhMOV7fjT4IJ7b9lk9638ONU%2FC%2FYc7cteZSiv0wSnFlcIgPeWj02eWkAQqTkec0BZlFthuQZsyYkdCGjBnem7cNxL9ZpzE%2BAfcSkaYh9vuOAYOKvnVhG5kzMHC%2BLnOqf2r4RcTZFCItqNkhgg%2B%2FD6qaixc6CcklFOpVloMpz2PKbnKYfakmp2W5cXQ%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 07:41:43,074 ERROR   wandb-upload_1:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': 'fLcsYHDHsXyGDmFDFhUJdA==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '4296'}
2023-09-02 07:41:43,074 ERROR   wandb-upload_1:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 07:41:47,309 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:41:52,312 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:41:57,314 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:41:59,113 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/94466346/557270874/e90691de0f334b62cf2d8f7f82eceb55?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=iR4Tf%2B1vZGLNCdRl1e3cfjqlxI%2B1dcNzWEshrYfBKXnmVYhhEfWoPLsdoLxCFjhHdLM8cTsm%2BCxyGydq%2BEhbIokMyk7JxGY3Q35V3HBtvAZyo6xXWnM5VM2Kl5LvN%2FyGaglS3SA5RHsAAV%2FuZKuIV4715YnvvOgpxoP21AmHjLkTlcVGnalPJ6A4bFSuOHH1AoDt1XwophdYlI5pGTLuI7%2Fujz%2BhIOxhTX00CZtthObUmQ0QivqxKZrgarIQocEWVCm32%2FwUTeg1TD7QxKc%2FgV5TO7iXCpXtwpByOsHNByI3xCfuzLumSPkutWs6AOi0JxypzOvDXTMGlJpMWeqJNg%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/94466346/557270874/e90691de0f334b62cf2d8f7f82eceb55?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=iR4Tf%2B1vZGLNCdRl1e3cfjqlxI%2B1dcNzWEshrYfBKXnmVYhhEfWoPLsdoLxCFjhHdLM8cTsm%2BCxyGydq%2BEhbIokMyk7JxGY3Q35V3HBtvAZyo6xXWnM5VM2Kl5LvN%2FyGaglS3SA5RHsAAV%2FuZKuIV4715YnvvOgpxoP21AmHjLkTlcVGnalPJ6A4bFSuOHH1AoDt1XwophdYlI5pGTLuI7%2Fujz%2BhIOxhTX00CZtthObUmQ0QivqxKZrgarIQocEWVCm32%2FwUTeg1TD7QxKc%2FgV5TO7iXCpXtwpByOsHNByI3xCfuzLumSPkutWs6AOi0JxypzOvDXTMGlJpMWeqJNg%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 07:41:59,113 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': '6QaR3g8zS2LPLY9/guzrVQ==', 'Content-Type': 'application/json', 'Content-Length': '13956'}
2023-09-02 07:41:59,113 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 07:42:01,479 ERROR   wandb-upload_1:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/94466346/557270874/7cb72c6070c7b17c860e614316150974?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=dUD2Kxz1IQ8b6fM%2BN0AjL7Q%2B6uG0nfs4ITNHi%2Bl3oIMg0lyWyeFdrj%2FoAgvFVebnCKZHYwKm6oXsLGxf%2BnVxBNJcESbSeGqpb8g7yBg%2B5YuYeT80B5Ml5N4venmTnxBYvVCLauDN%2B%2B47WXgAQSkrlo77hmFYMhMOV7fjT4IJ7b9lk9638ONU%2FC%2FYc7cteZSiv0wSnFlcIgPeWj02eWkAQqTkec0BZlFthuQZsyYkdCGjBnem7cNxL9ZpzE%2BAfcSkaYh9vuOAYOKvnVhG5kzMHC%2BLnOqf2r4RcTZFCItqNkhgg%2B%2FD6qaixc6CcklFOpVloMpz2PKbnKYfakmp2W5cXQ%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/94466346/557270874/7cb72c6070c7b17c860e614316150974?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=dUD2Kxz1IQ8b6fM%2BN0AjL7Q%2B6uG0nfs4ITNHi%2Bl3oIMg0lyWyeFdrj%2FoAgvFVebnCKZHYwKm6oXsLGxf%2BnVxBNJcESbSeGqpb8g7yBg%2B5YuYeT80B5Ml5N4venmTnxBYvVCLauDN%2B%2B47WXgAQSkrlo77hmFYMhMOV7fjT4IJ7b9lk9638ONU%2FC%2FYc7cteZSiv0wSnFlcIgPeWj02eWkAQqTkec0BZlFthuQZsyYkdCGjBnem7cNxL9ZpzE%2BAfcSkaYh9vuOAYOKvnVhG5kzMHC%2BLnOqf2r4RcTZFCItqNkhgg%2B%2FD6qaixc6CcklFOpVloMpz2PKbnKYfakmp2W5cXQ%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 07:42:01,479 ERROR   wandb-upload_1:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': 'fLcsYHDHsXyGDmFDFhUJdA==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '4296'}
2023-09-02 07:42:01,479 ERROR   wandb-upload_1:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 07:42:02,316 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:42:07,318 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:42:12,320 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:42:17,322 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:42:22,324 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:42:27,326 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:42:32,328 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:42:37,329 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:42:38,959 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/94466346/557270874/e90691de0f334b62cf2d8f7f82eceb55?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=iR4Tf%2B1vZGLNCdRl1e3cfjqlxI%2B1dcNzWEshrYfBKXnmVYhhEfWoPLsdoLxCFjhHdLM8cTsm%2BCxyGydq%2BEhbIokMyk7JxGY3Q35V3HBtvAZyo6xXWnM5VM2Kl5LvN%2FyGaglS3SA5RHsAAV%2FuZKuIV4715YnvvOgpxoP21AmHjLkTlcVGnalPJ6A4bFSuOHH1AoDt1XwophdYlI5pGTLuI7%2Fujz%2BhIOxhTX00CZtthObUmQ0QivqxKZrgarIQocEWVCm32%2FwUTeg1TD7QxKc%2FgV5TO7iXCpXtwpByOsHNByI3xCfuzLumSPkutWs6AOi0JxypzOvDXTMGlJpMWeqJNg%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/94466346/557270874/e90691de0f334b62cf2d8f7f82eceb55?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=iR4Tf%2B1vZGLNCdRl1e3cfjqlxI%2B1dcNzWEshrYfBKXnmVYhhEfWoPLsdoLxCFjhHdLM8cTsm%2BCxyGydq%2BEhbIokMyk7JxGY3Q35V3HBtvAZyo6xXWnM5VM2Kl5LvN%2FyGaglS3SA5RHsAAV%2FuZKuIV4715YnvvOgpxoP21AmHjLkTlcVGnalPJ6A4bFSuOHH1AoDt1XwophdYlI5pGTLuI7%2Fujz%2BhIOxhTX00CZtthObUmQ0QivqxKZrgarIQocEWVCm32%2FwUTeg1TD7QxKc%2FgV5TO7iXCpXtwpByOsHNByI3xCfuzLumSPkutWs6AOi0JxypzOvDXTMGlJpMWeqJNg%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 07:42:38,959 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': '6QaR3g8zS2LPLY9/guzrVQ==', 'Content-Type': 'application/json', 'Content-Length': '13956'}
2023-09-02 07:42:38,960 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 07:42:39,353 ERROR   wandb-upload_1:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/94466346/557270874/7cb72c6070c7b17c860e614316150974?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=dUD2Kxz1IQ8b6fM%2BN0AjL7Q%2B6uG0nfs4ITNHi%2Bl3oIMg0lyWyeFdrj%2FoAgvFVebnCKZHYwKm6oXsLGxf%2BnVxBNJcESbSeGqpb8g7yBg%2B5YuYeT80B5Ml5N4venmTnxBYvVCLauDN%2B%2B47WXgAQSkrlo77hmFYMhMOV7fjT4IJ7b9lk9638ONU%2FC%2FYc7cteZSiv0wSnFlcIgPeWj02eWkAQqTkec0BZlFthuQZsyYkdCGjBnem7cNxL9ZpzE%2BAfcSkaYh9vuOAYOKvnVhG5kzMHC%2BLnOqf2r4RcTZFCItqNkhgg%2B%2FD6qaixc6CcklFOpVloMpz2PKbnKYfakmp2W5cXQ%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/94466346/557270874/7cb72c6070c7b17c860e614316150974?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=dUD2Kxz1IQ8b6fM%2BN0AjL7Q%2B6uG0nfs4ITNHi%2Bl3oIMg0lyWyeFdrj%2FoAgvFVebnCKZHYwKm6oXsLGxf%2BnVxBNJcESbSeGqpb8g7yBg%2B5YuYeT80B5Ml5N4venmTnxBYvVCLauDN%2B%2B47WXgAQSkrlo77hmFYMhMOV7fjT4IJ7b9lk9638ONU%2FC%2FYc7cteZSiv0wSnFlcIgPeWj02eWkAQqTkec0BZlFthuQZsyYkdCGjBnem7cNxL9ZpzE%2BAfcSkaYh9vuOAYOKvnVhG5kzMHC%2BLnOqf2r4RcTZFCItqNkhgg%2B%2FD6qaixc6CcklFOpVloMpz2PKbnKYfakmp2W5cXQ%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 07:42:39,354 ERROR   wandb-upload_1:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': 'fLcsYHDHsXyGDmFDFhUJdA==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '4296'}
2023-09-02 07:42:39,354 ERROR   wandb-upload_1:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 07:42:42,331 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:42:47,333 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:42:52,335 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:42:57,337 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:43:02,339 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:43:07,341 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:43:12,343 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:43:17,344 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:43:22,346 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:43:27,348 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:43:32,350 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:43:37,352 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:43:42,355 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:43:46,413 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/94466346/557270874/e90691de0f334b62cf2d8f7f82eceb55?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=iR4Tf%2B1vZGLNCdRl1e3cfjqlxI%2B1dcNzWEshrYfBKXnmVYhhEfWoPLsdoLxCFjhHdLM8cTsm%2BCxyGydq%2BEhbIokMyk7JxGY3Q35V3HBtvAZyo6xXWnM5VM2Kl5LvN%2FyGaglS3SA5RHsAAV%2FuZKuIV4715YnvvOgpxoP21AmHjLkTlcVGnalPJ6A4bFSuOHH1AoDt1XwophdYlI5pGTLuI7%2Fujz%2BhIOxhTX00CZtthObUmQ0QivqxKZrgarIQocEWVCm32%2FwUTeg1TD7QxKc%2FgV5TO7iXCpXtwpByOsHNByI3xCfuzLumSPkutWs6AOi0JxypzOvDXTMGlJpMWeqJNg%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/94466346/557270874/e90691de0f334b62cf2d8f7f82eceb55?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=iR4Tf%2B1vZGLNCdRl1e3cfjqlxI%2B1dcNzWEshrYfBKXnmVYhhEfWoPLsdoLxCFjhHdLM8cTsm%2BCxyGydq%2BEhbIokMyk7JxGY3Q35V3HBtvAZyo6xXWnM5VM2Kl5LvN%2FyGaglS3SA5RHsAAV%2FuZKuIV4715YnvvOgpxoP21AmHjLkTlcVGnalPJ6A4bFSuOHH1AoDt1XwophdYlI5pGTLuI7%2Fujz%2BhIOxhTX00CZtthObUmQ0QivqxKZrgarIQocEWVCm32%2FwUTeg1TD7QxKc%2FgV5TO7iXCpXtwpByOsHNByI3xCfuzLumSPkutWs6AOi0JxypzOvDXTMGlJpMWeqJNg%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 07:43:46,414 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': '6QaR3g8zS2LPLY9/guzrVQ==', 'Content-Type': 'application/json', 'Content-Length': '13956'}
2023-09-02 07:43:46,414 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 07:43:47,362 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:43:52,364 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:43:56,769 ERROR   wandb-upload_1:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/94466346/557270874/7cb72c6070c7b17c860e614316150974?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=dUD2Kxz1IQ8b6fM%2BN0AjL7Q%2B6uG0nfs4ITNHi%2Bl3oIMg0lyWyeFdrj%2FoAgvFVebnCKZHYwKm6oXsLGxf%2BnVxBNJcESbSeGqpb8g7yBg%2B5YuYeT80B5Ml5N4venmTnxBYvVCLauDN%2B%2B47WXgAQSkrlo77hmFYMhMOV7fjT4IJ7b9lk9638ONU%2FC%2FYc7cteZSiv0wSnFlcIgPeWj02eWkAQqTkec0BZlFthuQZsyYkdCGjBnem7cNxL9ZpzE%2BAfcSkaYh9vuOAYOKvnVhG5kzMHC%2BLnOqf2r4RcTZFCItqNkhgg%2B%2FD6qaixc6CcklFOpVloMpz2PKbnKYfakmp2W5cXQ%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/94466346/557270874/7cb72c6070c7b17c860e614316150974?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=dUD2Kxz1IQ8b6fM%2BN0AjL7Q%2B6uG0nfs4ITNHi%2Bl3oIMg0lyWyeFdrj%2FoAgvFVebnCKZHYwKm6oXsLGxf%2BnVxBNJcESbSeGqpb8g7yBg%2B5YuYeT80B5Ml5N4venmTnxBYvVCLauDN%2B%2B47WXgAQSkrlo77hmFYMhMOV7fjT4IJ7b9lk9638ONU%2FC%2FYc7cteZSiv0wSnFlcIgPeWj02eWkAQqTkec0BZlFthuQZsyYkdCGjBnem7cNxL9ZpzE%2BAfcSkaYh9vuOAYOKvnVhG5kzMHC%2BLnOqf2r4RcTZFCItqNkhgg%2B%2FD6qaixc6CcklFOpVloMpz2PKbnKYfakmp2W5cXQ%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 07:43:56,769 ERROR   wandb-upload_1:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': 'fLcsYHDHsXyGDmFDFhUJdA==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '4296'}
2023-09-02 07:43:56,770 ERROR   wandb-upload_1:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 07:43:57,366 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:44:02,368 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:44:07,369 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:44:12,371 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:44:17,374 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:44:22,376 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:44:27,378 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:44:32,380 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:44:37,382 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:44:42,384 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:44:47,386 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:44:52,389 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:44:57,390 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:45:02,392 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:45:07,394 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:45:12,396 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:45:17,397 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:45:22,399 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:45:27,401 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:45:32,403 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:45:37,405 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:45:42,407 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:45:47,408 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:45:52,410 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:45:54,845 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/ucl_2023_com00087/trlx/an06fkb2/wandb-metadata.json?Expires=1693697418&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IyJ78E4KYvK780tDzzhndQFjwFadOPGOOCxnnEhzBCNSc7E8DaWyD7Lp%2FjsL4oWwssBH%2FxAkxObYd9Ui3BBe1qGBk3FBeebmV%2BvnaQayanzemeyJJ%2BmoiQ%2BGrkoM37rmz05qOw%2BVSOV9tAi1btjqK0w%2BibuxM3fIy1LvGk7wKOt61kqykMvq15jBIZogsrTY4aqjRbNU2Ynxr0dFwLwNagX07gZIpvze5lDOvj0bxGCSIf9cUqZg4rHwuS5UCmugAbtjVuoHbcXtPoZscv4XGd0EvURQ6kDGAfSgZnLVvEf6ITTyxaRoY1V427sqctkNNzOP%2Fn9xlgSqxJFQrhtVuA%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/ucl_2023_com00087/trlx/an06fkb2/wandb-metadata.json?Expires=1693697418&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IyJ78E4KYvK780tDzzhndQFjwFadOPGOOCxnnEhzBCNSc7E8DaWyD7Lp%2FjsL4oWwssBH%2FxAkxObYd9Ui3BBe1qGBk3FBeebmV%2BvnaQayanzemeyJJ%2BmoiQ%2BGrkoM37rmz05qOw%2BVSOV9tAi1btjqK0w%2BibuxM3fIy1LvGk7wKOt61kqykMvq15jBIZogsrTY4aqjRbNU2Ynxr0dFwLwNagX07gZIpvze5lDOvj0bxGCSIf9cUqZg4rHwuS5UCmugAbtjVuoHbcXtPoZscv4XGd0EvURQ6kDGAfSgZnLVvEf6ITTyxaRoY1V427sqctkNNzOP%2Fn9xlgSqxJFQrhtVuA%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 07:45:54,845 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '9452'}
2023-09-02 07:45:54,845 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 07:45:54,855 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/94466346/557270874/e90691de0f334b62cf2d8f7f82eceb55?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=iR4Tf%2B1vZGLNCdRl1e3cfjqlxI%2B1dcNzWEshrYfBKXnmVYhhEfWoPLsdoLxCFjhHdLM8cTsm%2BCxyGydq%2BEhbIokMyk7JxGY3Q35V3HBtvAZyo6xXWnM5VM2Kl5LvN%2FyGaglS3SA5RHsAAV%2FuZKuIV4715YnvvOgpxoP21AmHjLkTlcVGnalPJ6A4bFSuOHH1AoDt1XwophdYlI5pGTLuI7%2Fujz%2BhIOxhTX00CZtthObUmQ0QivqxKZrgarIQocEWVCm32%2FwUTeg1TD7QxKc%2FgV5TO7iXCpXtwpByOsHNByI3xCfuzLumSPkutWs6AOi0JxypzOvDXTMGlJpMWeqJNg%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/94466346/557270874/e90691de0f334b62cf2d8f7f82eceb55?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=iR4Tf%2B1vZGLNCdRl1e3cfjqlxI%2B1dcNzWEshrYfBKXnmVYhhEfWoPLsdoLxCFjhHdLM8cTsm%2BCxyGydq%2BEhbIokMyk7JxGY3Q35V3HBtvAZyo6xXWnM5VM2Kl5LvN%2FyGaglS3SA5RHsAAV%2FuZKuIV4715YnvvOgpxoP21AmHjLkTlcVGnalPJ6A4bFSuOHH1AoDt1XwophdYlI5pGTLuI7%2Fujz%2BhIOxhTX00CZtthObUmQ0QivqxKZrgarIQocEWVCm32%2FwUTeg1TD7QxKc%2FgV5TO7iXCpXtwpByOsHNByI3xCfuzLumSPkutWs6AOi0JxypzOvDXTMGlJpMWeqJNg%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 07:45:54,855 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': '6QaR3g8zS2LPLY9/guzrVQ==', 'Content-Type': 'application/json', 'Content-Length': '13956'}
2023-09-02 07:45:54,855 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 07:45:57,412 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:46:02,414 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:46:07,416 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:46:12,418 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:46:17,420 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:46:21,088 ERROR   wandb-upload_1:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/94466346/557270874/7cb72c6070c7b17c860e614316150974?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=dUD2Kxz1IQ8b6fM%2BN0AjL7Q%2B6uG0nfs4ITNHi%2Bl3oIMg0lyWyeFdrj%2FoAgvFVebnCKZHYwKm6oXsLGxf%2BnVxBNJcESbSeGqpb8g7yBg%2B5YuYeT80B5Ml5N4venmTnxBYvVCLauDN%2B%2B47WXgAQSkrlo77hmFYMhMOV7fjT4IJ7b9lk9638ONU%2FC%2FYc7cteZSiv0wSnFlcIgPeWj02eWkAQqTkec0BZlFthuQZsyYkdCGjBnem7cNxL9ZpzE%2BAfcSkaYh9vuOAYOKvnVhG5kzMHC%2BLnOqf2r4RcTZFCItqNkhgg%2B%2FD6qaixc6CcklFOpVloMpz2PKbnKYfakmp2W5cXQ%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/94466346/557270874/7cb72c6070c7b17c860e614316150974?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=dUD2Kxz1IQ8b6fM%2BN0AjL7Q%2B6uG0nfs4ITNHi%2Bl3oIMg0lyWyeFdrj%2FoAgvFVebnCKZHYwKm6oXsLGxf%2BnVxBNJcESbSeGqpb8g7yBg%2B5YuYeT80B5Ml5N4venmTnxBYvVCLauDN%2B%2B47WXgAQSkrlo77hmFYMhMOV7fjT4IJ7b9lk9638ONU%2FC%2FYc7cteZSiv0wSnFlcIgPeWj02eWkAQqTkec0BZlFthuQZsyYkdCGjBnem7cNxL9ZpzE%2BAfcSkaYh9vuOAYOKvnVhG5kzMHC%2BLnOqf2r4RcTZFCItqNkhgg%2B%2FD6qaixc6CcklFOpVloMpz2PKbnKYfakmp2W5cXQ%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 07:46:21,090 ERROR   wandb-upload_1:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': 'fLcsYHDHsXyGDmFDFhUJdA==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '4296'}
2023-09-02 07:46:21,090 ERROR   wandb-upload_1:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 07:46:22,422 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:46:27,424 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:46:32,426 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:46:37,428 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:46:42,429 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:46:47,431 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:46:52,433 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:46:57,435 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:47:02,437 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:47:07,439 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:47:12,441 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:47:17,443 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:47:22,445 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:47:27,447 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:47:32,449 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:47:37,451 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:47:42,453 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:47:47,455 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:47:52,457 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:47:57,459 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:48:02,461 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:48:07,463 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:48:12,465 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:48:17,467 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:48:22,469 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:48:27,470 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:48:32,472 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:48:37,474 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:48:42,477 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:48:47,478 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:48:52,480 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:48:57,482 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:49:02,484 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:49:07,486 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:49:12,488 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:49:17,490 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:49:22,492 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:49:27,494 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:49:32,496 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:49:37,497 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:49:42,499 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:49:47,501 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:49:52,503 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:49:57,505 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:50:02,506 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:50:07,508 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:50:12,510 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:50:17,512 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:50:22,514 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:50:25,679 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/94466346/557270874/e90691de0f334b62cf2d8f7f82eceb55?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=iR4Tf%2B1vZGLNCdRl1e3cfjqlxI%2B1dcNzWEshrYfBKXnmVYhhEfWoPLsdoLxCFjhHdLM8cTsm%2BCxyGydq%2BEhbIokMyk7JxGY3Q35V3HBtvAZyo6xXWnM5VM2Kl5LvN%2FyGaglS3SA5RHsAAV%2FuZKuIV4715YnvvOgpxoP21AmHjLkTlcVGnalPJ6A4bFSuOHH1AoDt1XwophdYlI5pGTLuI7%2Fujz%2BhIOxhTX00CZtthObUmQ0QivqxKZrgarIQocEWVCm32%2FwUTeg1TD7QxKc%2FgV5TO7iXCpXtwpByOsHNByI3xCfuzLumSPkutWs6AOi0JxypzOvDXTMGlJpMWeqJNg%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/94466346/557270874/e90691de0f334b62cf2d8f7f82eceb55?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=iR4Tf%2B1vZGLNCdRl1e3cfjqlxI%2B1dcNzWEshrYfBKXnmVYhhEfWoPLsdoLxCFjhHdLM8cTsm%2BCxyGydq%2BEhbIokMyk7JxGY3Q35V3HBtvAZyo6xXWnM5VM2Kl5LvN%2FyGaglS3SA5RHsAAV%2FuZKuIV4715YnvvOgpxoP21AmHjLkTlcVGnalPJ6A4bFSuOHH1AoDt1XwophdYlI5pGTLuI7%2Fujz%2BhIOxhTX00CZtthObUmQ0QivqxKZrgarIQocEWVCm32%2FwUTeg1TD7QxKc%2FgV5TO7iXCpXtwpByOsHNByI3xCfuzLumSPkutWs6AOi0JxypzOvDXTMGlJpMWeqJNg%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 07:50:25,679 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': '6QaR3g8zS2LPLY9/guzrVQ==', 'Content-Type': 'application/json', 'Content-Length': '13956'}
2023-09-02 07:50:25,679 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 07:50:27,516 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:50:32,518 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:50:37,519 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:50:42,521 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:50:47,523 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:50:52,525 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:50:57,527 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:51:02,529 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:51:07,530 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:51:12,532 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:51:17,534 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:51:22,537 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:51:25,805 ERROR   wandb-upload_1:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/94466346/557270874/7cb72c6070c7b17c860e614316150974?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=dUD2Kxz1IQ8b6fM%2BN0AjL7Q%2B6uG0nfs4ITNHi%2Bl3oIMg0lyWyeFdrj%2FoAgvFVebnCKZHYwKm6oXsLGxf%2BnVxBNJcESbSeGqpb8g7yBg%2B5YuYeT80B5Ml5N4venmTnxBYvVCLauDN%2B%2B47WXgAQSkrlo77hmFYMhMOV7fjT4IJ7b9lk9638ONU%2FC%2FYc7cteZSiv0wSnFlcIgPeWj02eWkAQqTkec0BZlFthuQZsyYkdCGjBnem7cNxL9ZpzE%2BAfcSkaYh9vuOAYOKvnVhG5kzMHC%2BLnOqf2r4RcTZFCItqNkhgg%2B%2FD6qaixc6CcklFOpVloMpz2PKbnKYfakmp2W5cXQ%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/94466346/557270874/7cb72c6070c7b17c860e614316150974?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=dUD2Kxz1IQ8b6fM%2BN0AjL7Q%2B6uG0nfs4ITNHi%2Bl3oIMg0lyWyeFdrj%2FoAgvFVebnCKZHYwKm6oXsLGxf%2BnVxBNJcESbSeGqpb8g7yBg%2B5YuYeT80B5Ml5N4venmTnxBYvVCLauDN%2B%2B47WXgAQSkrlo77hmFYMhMOV7fjT4IJ7b9lk9638ONU%2FC%2FYc7cteZSiv0wSnFlcIgPeWj02eWkAQqTkec0BZlFthuQZsyYkdCGjBnem7cNxL9ZpzE%2BAfcSkaYh9vuOAYOKvnVhG5kzMHC%2BLnOqf2r4RcTZFCItqNkhgg%2B%2FD6qaixc6CcklFOpVloMpz2PKbnKYfakmp2W5cXQ%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 07:51:25,805 ERROR   wandb-upload_1:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': 'fLcsYHDHsXyGDmFDFhUJdA==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '4296'}
2023-09-02 07:51:25,805 ERROR   wandb-upload_1:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 07:51:27,538 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:51:32,540 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:51:37,542 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:51:42,544 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:51:47,546 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:51:52,548 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:51:56,442 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/ucl_2023_com00087/trlx/an06fkb2/wandb-metadata.json?Expires=1693697418&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IyJ78E4KYvK780tDzzhndQFjwFadOPGOOCxnnEhzBCNSc7E8DaWyD7Lp%2FjsL4oWwssBH%2FxAkxObYd9Ui3BBe1qGBk3FBeebmV%2BvnaQayanzemeyJJ%2BmoiQ%2BGrkoM37rmz05qOw%2BVSOV9tAi1btjqK0w%2BibuxM3fIy1LvGk7wKOt61kqykMvq15jBIZogsrTY4aqjRbNU2Ynxr0dFwLwNagX07gZIpvze5lDOvj0bxGCSIf9cUqZg4rHwuS5UCmugAbtjVuoHbcXtPoZscv4XGd0EvURQ6kDGAfSgZnLVvEf6ITTyxaRoY1V427sqctkNNzOP%2Fn9xlgSqxJFQrhtVuA%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/ucl_2023_com00087/trlx/an06fkb2/wandb-metadata.json?Expires=1693697418&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IyJ78E4KYvK780tDzzhndQFjwFadOPGOOCxnnEhzBCNSc7E8DaWyD7Lp%2FjsL4oWwssBH%2FxAkxObYd9Ui3BBe1qGBk3FBeebmV%2BvnaQayanzemeyJJ%2BmoiQ%2BGrkoM37rmz05qOw%2BVSOV9tAi1btjqK0w%2BibuxM3fIy1LvGk7wKOt61kqykMvq15jBIZogsrTY4aqjRbNU2Ynxr0dFwLwNagX07gZIpvze5lDOvj0bxGCSIf9cUqZg4rHwuS5UCmugAbtjVuoHbcXtPoZscv4XGd0EvURQ6kDGAfSgZnLVvEf6ITTyxaRoY1V427sqctkNNzOP%2Fn9xlgSqxJFQrhtVuA%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 07:51:56,442 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '9452'}
2023-09-02 07:51:56,442 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 07:51:57,549 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:52:02,551 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:52:07,554 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:52:12,556 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:52:17,558 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:52:22,560 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:52:27,562 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:52:32,564 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:52:37,566 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:52:42,568 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:52:47,569 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:52:52,571 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:52:57,574 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:53:02,576 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:53:07,577 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:53:12,579 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:53:17,581 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:53:22,583 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:53:27,585 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:53:32,587 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:53:37,588 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:53:42,590 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:53:47,592 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:53:52,594 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:53:57,596 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:54:02,598 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:54:07,599 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:54:12,601 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:54:17,603 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:54:22,605 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:54:27,607 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:54:32,608 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:54:37,610 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:54:42,612 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:54:47,614 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:54:52,616 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:54:57,618 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:55:02,620 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:55:07,622 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:55:12,625 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:55:17,626 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:55:22,628 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:55:27,630 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:55:32,632 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:55:37,634 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:55:42,636 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:55:47,638 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:55:52,640 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:55:57,641 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:56:02,643 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:56:07,645 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:56:08,078 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/94466346/557270874/e90691de0f334b62cf2d8f7f82eceb55?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=iR4Tf%2B1vZGLNCdRl1e3cfjqlxI%2B1dcNzWEshrYfBKXnmVYhhEfWoPLsdoLxCFjhHdLM8cTsm%2BCxyGydq%2BEhbIokMyk7JxGY3Q35V3HBtvAZyo6xXWnM5VM2Kl5LvN%2FyGaglS3SA5RHsAAV%2FuZKuIV4715YnvvOgpxoP21AmHjLkTlcVGnalPJ6A4bFSuOHH1AoDt1XwophdYlI5pGTLuI7%2Fujz%2BhIOxhTX00CZtthObUmQ0QivqxKZrgarIQocEWVCm32%2FwUTeg1TD7QxKc%2FgV5TO7iXCpXtwpByOsHNByI3xCfuzLumSPkutWs6AOi0JxypzOvDXTMGlJpMWeqJNg%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/94466346/557270874/e90691de0f334b62cf2d8f7f82eceb55?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=iR4Tf%2B1vZGLNCdRl1e3cfjqlxI%2B1dcNzWEshrYfBKXnmVYhhEfWoPLsdoLxCFjhHdLM8cTsm%2BCxyGydq%2BEhbIokMyk7JxGY3Q35V3HBtvAZyo6xXWnM5VM2Kl5LvN%2FyGaglS3SA5RHsAAV%2FuZKuIV4715YnvvOgpxoP21AmHjLkTlcVGnalPJ6A4bFSuOHH1AoDt1XwophdYlI5pGTLuI7%2Fujz%2BhIOxhTX00CZtthObUmQ0QivqxKZrgarIQocEWVCm32%2FwUTeg1TD7QxKc%2FgV5TO7iXCpXtwpByOsHNByI3xCfuzLumSPkutWs6AOi0JxypzOvDXTMGlJpMWeqJNg%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 07:56:08,078 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': '6QaR3g8zS2LPLY9/guzrVQ==', 'Content-Type': 'application/json', 'Content-Length': '13956'}
2023-09-02 07:56:08,078 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 07:56:12,647 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:56:17,649 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:56:22,651 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:56:27,653 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:56:32,655 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:56:37,656 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:56:42,659 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:56:47,660 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:56:48,184 ERROR   wandb-upload_1:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/94466346/557270874/7cb72c6070c7b17c860e614316150974?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=dUD2Kxz1IQ8b6fM%2BN0AjL7Q%2B6uG0nfs4ITNHi%2Bl3oIMg0lyWyeFdrj%2FoAgvFVebnCKZHYwKm6oXsLGxf%2BnVxBNJcESbSeGqpb8g7yBg%2B5YuYeT80B5Ml5N4venmTnxBYvVCLauDN%2B%2B47WXgAQSkrlo77hmFYMhMOV7fjT4IJ7b9lk9638ONU%2FC%2FYc7cteZSiv0wSnFlcIgPeWj02eWkAQqTkec0BZlFthuQZsyYkdCGjBnem7cNxL9ZpzE%2BAfcSkaYh9vuOAYOKvnVhG5kzMHC%2BLnOqf2r4RcTZFCItqNkhgg%2B%2FD6qaixc6CcklFOpVloMpz2PKbnKYfakmp2W5cXQ%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/94466346/557270874/7cb72c6070c7b17c860e614316150974?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=dUD2Kxz1IQ8b6fM%2BN0AjL7Q%2B6uG0nfs4ITNHi%2Bl3oIMg0lyWyeFdrj%2FoAgvFVebnCKZHYwKm6oXsLGxf%2BnVxBNJcESbSeGqpb8g7yBg%2B5YuYeT80B5Ml5N4venmTnxBYvVCLauDN%2B%2B47WXgAQSkrlo77hmFYMhMOV7fjT4IJ7b9lk9638ONU%2FC%2FYc7cteZSiv0wSnFlcIgPeWj02eWkAQqTkec0BZlFthuQZsyYkdCGjBnem7cNxL9ZpzE%2BAfcSkaYh9vuOAYOKvnVhG5kzMHC%2BLnOqf2r4RcTZFCItqNkhgg%2B%2FD6qaixc6CcklFOpVloMpz2PKbnKYfakmp2W5cXQ%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 07:56:48,184 ERROR   wandb-upload_1:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': 'fLcsYHDHsXyGDmFDFhUJdA==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '4296'}
2023-09-02 07:56:48,184 ERROR   wandb-upload_1:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 07:56:52,662 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:56:57,664 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:57:02,666 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:57:07,668 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:57:12,670 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:57:17,672 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:57:22,674 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:57:27,676 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:57:32,678 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:57:37,680 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:57:42,682 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:57:47,684 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:57:51,482 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/ucl_2023_com00087/trlx/an06fkb2/wandb-metadata.json?Expires=1693697418&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IyJ78E4KYvK780tDzzhndQFjwFadOPGOOCxnnEhzBCNSc7E8DaWyD7Lp%2FjsL4oWwssBH%2FxAkxObYd9Ui3BBe1qGBk3FBeebmV%2BvnaQayanzemeyJJ%2BmoiQ%2BGrkoM37rmz05qOw%2BVSOV9tAi1btjqK0w%2BibuxM3fIy1LvGk7wKOt61kqykMvq15jBIZogsrTY4aqjRbNU2Ynxr0dFwLwNagX07gZIpvze5lDOvj0bxGCSIf9cUqZg4rHwuS5UCmugAbtjVuoHbcXtPoZscv4XGd0EvURQ6kDGAfSgZnLVvEf6ITTyxaRoY1V427sqctkNNzOP%2Fn9xlgSqxJFQrhtVuA%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/ucl_2023_com00087/trlx/an06fkb2/wandb-metadata.json?Expires=1693697418&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IyJ78E4KYvK780tDzzhndQFjwFadOPGOOCxnnEhzBCNSc7E8DaWyD7Lp%2FjsL4oWwssBH%2FxAkxObYd9Ui3BBe1qGBk3FBeebmV%2BvnaQayanzemeyJJ%2BmoiQ%2BGrkoM37rmz05qOw%2BVSOV9tAi1btjqK0w%2BibuxM3fIy1LvGk7wKOt61kqykMvq15jBIZogsrTY4aqjRbNU2Ynxr0dFwLwNagX07gZIpvze5lDOvj0bxGCSIf9cUqZg4rHwuS5UCmugAbtjVuoHbcXtPoZscv4XGd0EvURQ6kDGAfSgZnLVvEf6ITTyxaRoY1V427sqctkNNzOP%2Fn9xlgSqxJFQrhtVuA%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 07:57:51,482 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '9452'}
2023-09-02 07:57:51,482 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 07:57:52,686 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:57:57,688 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:58:02,690 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:58:07,691 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:58:12,693 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:58:17,695 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:58:22,697 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:58:27,699 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:58:32,701 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:58:37,702 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:58:42,704 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:58:47,706 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:58:52,709 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:58:57,711 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:59:02,713 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:59:07,715 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:59:12,717 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:59:17,719 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:59:22,720 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:59:27,722 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:59:32,724 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:59:37,726 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:59:42,728 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:59:47,730 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:59:52,732 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 07:59:57,734 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 08:00:02,736 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 08:00:07,738 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 08:00:12,741 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 08:00:17,743 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 08:00:22,745 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 08:00:27,747 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 08:00:32,749 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 08:00:37,751 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 08:00:42,753 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 08:00:47,755 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 08:00:52,757 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 08:00:57,758 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 08:01:02,760 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 08:01:07,762 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 08:01:12,764 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 08:01:17,766 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 08:01:22,768 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 08:01:27,770 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 08:01:32,772 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 08:01:37,774 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 08:01:42,776 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 08:01:47,777 DEBUG   HandlerThread:9010 [handler.py:handle_request():144] handle_request: keepalive
2023-09-02 08:01:51,495 WARNING StreamThr :9010 [internal.py:is_dead():414] Internal process exiting, parent pid 8916 disappeared
2023-09-02 08:01:51,496 ERROR   StreamThr :9010 [internal.py:wandb_internal():152] Internal process shutdown.
2023-09-02 08:01:51,779 INFO    HandlerThread:9010 [handler.py:finish():845] shutting down handler
2023-09-02 08:01:52,132 INFO    WriterThread:9010 [datastore.py:close():298] close: /root/autodl-tmp/msc_ml/llm_thought/Train/SVAMP/wandb/run-20230902_073012-an06fkb2/run-an06fkb2.wandb
2023-09-02 08:01:52,320 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/94466346/557270874/e90691de0f334b62cf2d8f7f82eceb55?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=iR4Tf%2B1vZGLNCdRl1e3cfjqlxI%2B1dcNzWEshrYfBKXnmVYhhEfWoPLsdoLxCFjhHdLM8cTsm%2BCxyGydq%2BEhbIokMyk7JxGY3Q35V3HBtvAZyo6xXWnM5VM2Kl5LvN%2FyGaglS3SA5RHsAAV%2FuZKuIV4715YnvvOgpxoP21AmHjLkTlcVGnalPJ6A4bFSuOHH1AoDt1XwophdYlI5pGTLuI7%2Fujz%2BhIOxhTX00CZtthObUmQ0QivqxKZrgarIQocEWVCm32%2FwUTeg1TD7QxKc%2FgV5TO7iXCpXtwpByOsHNByI3xCfuzLumSPkutWs6AOi0JxypzOvDXTMGlJpMWeqJNg%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/94466346/557270874/e90691de0f334b62cf2d8f7f82eceb55?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=iR4Tf%2B1vZGLNCdRl1e3cfjqlxI%2B1dcNzWEshrYfBKXnmVYhhEfWoPLsdoLxCFjhHdLM8cTsm%2BCxyGydq%2BEhbIokMyk7JxGY3Q35V3HBtvAZyo6xXWnM5VM2Kl5LvN%2FyGaglS3SA5RHsAAV%2FuZKuIV4715YnvvOgpxoP21AmHjLkTlcVGnalPJ6A4bFSuOHH1AoDt1XwophdYlI5pGTLuI7%2Fujz%2BhIOxhTX00CZtthObUmQ0QivqxKZrgarIQocEWVCm32%2FwUTeg1TD7QxKc%2FgV5TO7iXCpXtwpByOsHNByI3xCfuzLumSPkutWs6AOi0JxypzOvDXTMGlJpMWeqJNg%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 08:01:52,320 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': '6QaR3g8zS2LPLY9/guzrVQ==', 'Content-Type': 'application/json', 'Content-Length': '13956'}
2023-09-02 08:01:52,320 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 08:02:40,699 ERROR   wandb-upload_1:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/94466346/557270874/7cb72c6070c7b17c860e614316150974?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=dUD2Kxz1IQ8b6fM%2BN0AjL7Q%2B6uG0nfs4ITNHi%2Bl3oIMg0lyWyeFdrj%2FoAgvFVebnCKZHYwKm6oXsLGxf%2BnVxBNJcESbSeGqpb8g7yBg%2B5YuYeT80B5Ml5N4venmTnxBYvVCLauDN%2B%2B47WXgAQSkrlo77hmFYMhMOV7fjT4IJ7b9lk9638ONU%2FC%2FYc7cteZSiv0wSnFlcIgPeWj02eWkAQqTkec0BZlFthuQZsyYkdCGjBnem7cNxL9ZpzE%2BAfcSkaYh9vuOAYOKvnVhG5kzMHC%2BLnOqf2r4RcTZFCItqNkhgg%2B%2FD6qaixc6CcklFOpVloMpz2PKbnKYfakmp2W5cXQ%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/94466346/557270874/7cb72c6070c7b17c860e614316150974?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=dUD2Kxz1IQ8b6fM%2BN0AjL7Q%2B6uG0nfs4ITNHi%2Bl3oIMg0lyWyeFdrj%2FoAgvFVebnCKZHYwKm6oXsLGxf%2BnVxBNJcESbSeGqpb8g7yBg%2B5YuYeT80B5Ml5N4venmTnxBYvVCLauDN%2B%2B47WXgAQSkrlo77hmFYMhMOV7fjT4IJ7b9lk9638ONU%2FC%2FYc7cteZSiv0wSnFlcIgPeWj02eWkAQqTkec0BZlFthuQZsyYkdCGjBnem7cNxL9ZpzE%2BAfcSkaYh9vuOAYOKvnVhG5kzMHC%2BLnOqf2r4RcTZFCItqNkhgg%2B%2FD6qaixc6CcklFOpVloMpz2PKbnKYfakmp2W5cXQ%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 08:02:40,699 ERROR   wandb-upload_1:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': 'fLcsYHDHsXyGDmFDFhUJdA==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '4296'}
2023-09-02 08:02:40,700 ERROR   wandb-upload_1:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 08:03:59,142 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/ucl_2023_com00087/trlx/an06fkb2/wandb-metadata.json?Expires=1693697418&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IyJ78E4KYvK780tDzzhndQFjwFadOPGOOCxnnEhzBCNSc7E8DaWyD7Lp%2FjsL4oWwssBH%2FxAkxObYd9Ui3BBe1qGBk3FBeebmV%2BvnaQayanzemeyJJ%2BmoiQ%2BGrkoM37rmz05qOw%2BVSOV9tAi1btjqK0w%2BibuxM3fIy1LvGk7wKOt61kqykMvq15jBIZogsrTY4aqjRbNU2Ynxr0dFwLwNagX07gZIpvze5lDOvj0bxGCSIf9cUqZg4rHwuS5UCmugAbtjVuoHbcXtPoZscv4XGd0EvURQ6kDGAfSgZnLVvEf6ITTyxaRoY1V427sqctkNNzOP%2Fn9xlgSqxJFQrhtVuA%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/ucl_2023_com00087/trlx/an06fkb2/wandb-metadata.json?Expires=1693697418&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IyJ78E4KYvK780tDzzhndQFjwFadOPGOOCxnnEhzBCNSc7E8DaWyD7Lp%2FjsL4oWwssBH%2FxAkxObYd9Ui3BBe1qGBk3FBeebmV%2BvnaQayanzemeyJJ%2BmoiQ%2BGrkoM37rmz05qOw%2BVSOV9tAi1btjqK0w%2BibuxM3fIy1LvGk7wKOt61kqykMvq15jBIZogsrTY4aqjRbNU2Ynxr0dFwLwNagX07gZIpvze5lDOvj0bxGCSIf9cUqZg4rHwuS5UCmugAbtjVuoHbcXtPoZscv4XGd0EvURQ6kDGAfSgZnLVvEf6ITTyxaRoY1V427sqctkNNzOP%2Fn9xlgSqxJFQrhtVuA%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 08:03:59,142 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '9452'}
2023-09-02 08:03:59,142 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 08:07:46,382 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/94466346/557270874/e90691de0f334b62cf2d8f7f82eceb55?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=iR4Tf%2B1vZGLNCdRl1e3cfjqlxI%2B1dcNzWEshrYfBKXnmVYhhEfWoPLsdoLxCFjhHdLM8cTsm%2BCxyGydq%2BEhbIokMyk7JxGY3Q35V3HBtvAZyo6xXWnM5VM2Kl5LvN%2FyGaglS3SA5RHsAAV%2FuZKuIV4715YnvvOgpxoP21AmHjLkTlcVGnalPJ6A4bFSuOHH1AoDt1XwophdYlI5pGTLuI7%2Fujz%2BhIOxhTX00CZtthObUmQ0QivqxKZrgarIQocEWVCm32%2FwUTeg1TD7QxKc%2FgV5TO7iXCpXtwpByOsHNByI3xCfuzLumSPkutWs6AOi0JxypzOvDXTMGlJpMWeqJNg%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/94466346/557270874/e90691de0f334b62cf2d8f7f82eceb55?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=iR4Tf%2B1vZGLNCdRl1e3cfjqlxI%2B1dcNzWEshrYfBKXnmVYhhEfWoPLsdoLxCFjhHdLM8cTsm%2BCxyGydq%2BEhbIokMyk7JxGY3Q35V3HBtvAZyo6xXWnM5VM2Kl5LvN%2FyGaglS3SA5RHsAAV%2FuZKuIV4715YnvvOgpxoP21AmHjLkTlcVGnalPJ6A4bFSuOHH1AoDt1XwophdYlI5pGTLuI7%2Fujz%2BhIOxhTX00CZtthObUmQ0QivqxKZrgarIQocEWVCm32%2FwUTeg1TD7QxKc%2FgV5TO7iXCpXtwpByOsHNByI3xCfuzLumSPkutWs6AOi0JxypzOvDXTMGlJpMWeqJNg%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 08:07:46,382 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': '6QaR3g8zS2LPLY9/guzrVQ==', 'Content-Type': 'application/json', 'Content-Length': '13956'}
2023-09-02 08:07:46,382 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 08:08:41,886 INFO    wandb-upload_1:9010 [upload_job.py:push():95] Uploaded file /root/.local/share/wandb/artifacts/staging/tmph7p377cu
2023-09-02 08:09:48,480 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/ucl_2023_com00087/trlx/an06fkb2/wandb-metadata.json?Expires=1693697418&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IyJ78E4KYvK780tDzzhndQFjwFadOPGOOCxnnEhzBCNSc7E8DaWyD7Lp%2FjsL4oWwssBH%2FxAkxObYd9Ui3BBe1qGBk3FBeebmV%2BvnaQayanzemeyJJ%2BmoiQ%2BGrkoM37rmz05qOw%2BVSOV9tAi1btjqK0w%2BibuxM3fIy1LvGk7wKOt61kqykMvq15jBIZogsrTY4aqjRbNU2Ynxr0dFwLwNagX07gZIpvze5lDOvj0bxGCSIf9cUqZg4rHwuS5UCmugAbtjVuoHbcXtPoZscv4XGd0EvURQ6kDGAfSgZnLVvEf6ITTyxaRoY1V427sqctkNNzOP%2Fn9xlgSqxJFQrhtVuA%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/ucl_2023_com00087/trlx/an06fkb2/wandb-metadata.json?Expires=1693697418&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IyJ78E4KYvK780tDzzhndQFjwFadOPGOOCxnnEhzBCNSc7E8DaWyD7Lp%2FjsL4oWwssBH%2FxAkxObYd9Ui3BBe1qGBk3FBeebmV%2BvnaQayanzemeyJJ%2BmoiQ%2BGrkoM37rmz05qOw%2BVSOV9tAi1btjqK0w%2BibuxM3fIy1LvGk7wKOt61kqykMvq15jBIZogsrTY4aqjRbNU2Ynxr0dFwLwNagX07gZIpvze5lDOvj0bxGCSIf9cUqZg4rHwuS5UCmugAbtjVuoHbcXtPoZscv4XGd0EvURQ6kDGAfSgZnLVvEf6ITTyxaRoY1V427sqctkNNzOP%2Fn9xlgSqxJFQrhtVuA%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 08:09:48,480 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '9452'}
2023-09-02 08:09:48,481 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 08:12:50,614 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/94466346/557270874/e90691de0f334b62cf2d8f7f82eceb55?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=iR4Tf%2B1vZGLNCdRl1e3cfjqlxI%2B1dcNzWEshrYfBKXnmVYhhEfWoPLsdoLxCFjhHdLM8cTsm%2BCxyGydq%2BEhbIokMyk7JxGY3Q35V3HBtvAZyo6xXWnM5VM2Kl5LvN%2FyGaglS3SA5RHsAAV%2FuZKuIV4715YnvvOgpxoP21AmHjLkTlcVGnalPJ6A4bFSuOHH1AoDt1XwophdYlI5pGTLuI7%2Fujz%2BhIOxhTX00CZtthObUmQ0QivqxKZrgarIQocEWVCm32%2FwUTeg1TD7QxKc%2FgV5TO7iXCpXtwpByOsHNByI3xCfuzLumSPkutWs6AOi0JxypzOvDXTMGlJpMWeqJNg%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/94466346/557270874/e90691de0f334b62cf2d8f7f82eceb55?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=iR4Tf%2B1vZGLNCdRl1e3cfjqlxI%2B1dcNzWEshrYfBKXnmVYhhEfWoPLsdoLxCFjhHdLM8cTsm%2BCxyGydq%2BEhbIokMyk7JxGY3Q35V3HBtvAZyo6xXWnM5VM2Kl5LvN%2FyGaglS3SA5RHsAAV%2FuZKuIV4715YnvvOgpxoP21AmHjLkTlcVGnalPJ6A4bFSuOHH1AoDt1XwophdYlI5pGTLuI7%2Fujz%2BhIOxhTX00CZtthObUmQ0QivqxKZrgarIQocEWVCm32%2FwUTeg1TD7QxKc%2FgV5TO7iXCpXtwpByOsHNByI3xCfuzLumSPkutWs6AOi0JxypzOvDXTMGlJpMWeqJNg%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 08:12:50,614 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': '6QaR3g8zS2LPLY9/guzrVQ==', 'Content-Type': 'application/json', 'Content-Length': '13956'}
2023-09-02 08:12:50,614 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 08:15:16,333 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/ucl_2023_com00087/trlx/an06fkb2/wandb-metadata.json?Expires=1693697418&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IyJ78E4KYvK780tDzzhndQFjwFadOPGOOCxnnEhzBCNSc7E8DaWyD7Lp%2FjsL4oWwssBH%2FxAkxObYd9Ui3BBe1qGBk3FBeebmV%2BvnaQayanzemeyJJ%2BmoiQ%2BGrkoM37rmz05qOw%2BVSOV9tAi1btjqK0w%2BibuxM3fIy1LvGk7wKOt61kqykMvq15jBIZogsrTY4aqjRbNU2Ynxr0dFwLwNagX07gZIpvze5lDOvj0bxGCSIf9cUqZg4rHwuS5UCmugAbtjVuoHbcXtPoZscv4XGd0EvURQ6kDGAfSgZnLVvEf6ITTyxaRoY1V427sqctkNNzOP%2Fn9xlgSqxJFQrhtVuA%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/ucl_2023_com00087/trlx/an06fkb2/wandb-metadata.json?Expires=1693697418&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IyJ78E4KYvK780tDzzhndQFjwFadOPGOOCxnnEhzBCNSc7E8DaWyD7Lp%2FjsL4oWwssBH%2FxAkxObYd9Ui3BBe1qGBk3FBeebmV%2BvnaQayanzemeyJJ%2BmoiQ%2BGrkoM37rmz05qOw%2BVSOV9tAi1btjqK0w%2BibuxM3fIy1LvGk7wKOt61kqykMvq15jBIZogsrTY4aqjRbNU2Ynxr0dFwLwNagX07gZIpvze5lDOvj0bxGCSIf9cUqZg4rHwuS5UCmugAbtjVuoHbcXtPoZscv4XGd0EvURQ6kDGAfSgZnLVvEf6ITTyxaRoY1V427sqctkNNzOP%2Fn9xlgSqxJFQrhtVuA%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 08:15:16,334 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '9452'}
2023-09-02 08:15:16,334 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 08:18:29,866 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/94466346/557270874/e90691de0f334b62cf2d8f7f82eceb55?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=iR4Tf%2B1vZGLNCdRl1e3cfjqlxI%2B1dcNzWEshrYfBKXnmVYhhEfWoPLsdoLxCFjhHdLM8cTsm%2BCxyGydq%2BEhbIokMyk7JxGY3Q35V3HBtvAZyo6xXWnM5VM2Kl5LvN%2FyGaglS3SA5RHsAAV%2FuZKuIV4715YnvvOgpxoP21AmHjLkTlcVGnalPJ6A4bFSuOHH1AoDt1XwophdYlI5pGTLuI7%2Fujz%2BhIOxhTX00CZtthObUmQ0QivqxKZrgarIQocEWVCm32%2FwUTeg1TD7QxKc%2FgV5TO7iXCpXtwpByOsHNByI3xCfuzLumSPkutWs6AOi0JxypzOvDXTMGlJpMWeqJNg%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/94466346/557270874/e90691de0f334b62cf2d8f7f82eceb55?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=iR4Tf%2B1vZGLNCdRl1e3cfjqlxI%2B1dcNzWEshrYfBKXnmVYhhEfWoPLsdoLxCFjhHdLM8cTsm%2BCxyGydq%2BEhbIokMyk7JxGY3Q35V3HBtvAZyo6xXWnM5VM2Kl5LvN%2FyGaglS3SA5RHsAAV%2FuZKuIV4715YnvvOgpxoP21AmHjLkTlcVGnalPJ6A4bFSuOHH1AoDt1XwophdYlI5pGTLuI7%2Fujz%2BhIOxhTX00CZtthObUmQ0QivqxKZrgarIQocEWVCm32%2FwUTeg1TD7QxKc%2FgV5TO7iXCpXtwpByOsHNByI3xCfuzLumSPkutWs6AOi0JxypzOvDXTMGlJpMWeqJNg%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 08:18:29,867 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': '6QaR3g8zS2LPLY9/guzrVQ==', 'Content-Type': 'application/json', 'Content-Length': '13956'}
2023-09-02 08:18:29,867 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 08:20:44,203 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/ucl_2023_com00087/trlx/an06fkb2/wandb-metadata.json?Expires=1693697418&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IyJ78E4KYvK780tDzzhndQFjwFadOPGOOCxnnEhzBCNSc7E8DaWyD7Lp%2FjsL4oWwssBH%2FxAkxObYd9Ui3BBe1qGBk3FBeebmV%2BvnaQayanzemeyJJ%2BmoiQ%2BGrkoM37rmz05qOw%2BVSOV9tAi1btjqK0w%2BibuxM3fIy1LvGk7wKOt61kqykMvq15jBIZogsrTY4aqjRbNU2Ynxr0dFwLwNagX07gZIpvze5lDOvj0bxGCSIf9cUqZg4rHwuS5UCmugAbtjVuoHbcXtPoZscv4XGd0EvURQ6kDGAfSgZnLVvEf6ITTyxaRoY1V427sqctkNNzOP%2Fn9xlgSqxJFQrhtVuA%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/ucl_2023_com00087/trlx/an06fkb2/wandb-metadata.json?Expires=1693697418&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IyJ78E4KYvK780tDzzhndQFjwFadOPGOOCxnnEhzBCNSc7E8DaWyD7Lp%2FjsL4oWwssBH%2FxAkxObYd9Ui3BBe1qGBk3FBeebmV%2BvnaQayanzemeyJJ%2BmoiQ%2BGrkoM37rmz05qOw%2BVSOV9tAi1btjqK0w%2BibuxM3fIy1LvGk7wKOt61kqykMvq15jBIZogsrTY4aqjRbNU2Ynxr0dFwLwNagX07gZIpvze5lDOvj0bxGCSIf9cUqZg4rHwuS5UCmugAbtjVuoHbcXtPoZscv4XGd0EvURQ6kDGAfSgZnLVvEf6ITTyxaRoY1V427sqctkNNzOP%2Fn9xlgSqxJFQrhtVuA%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 08:20:44,204 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '9452'}
2023-09-02 08:20:44,204 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 08:23:40,166 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/94466346/557270874/e90691de0f334b62cf2d8f7f82eceb55?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=iR4Tf%2B1vZGLNCdRl1e3cfjqlxI%2B1dcNzWEshrYfBKXnmVYhhEfWoPLsdoLxCFjhHdLM8cTsm%2BCxyGydq%2BEhbIokMyk7JxGY3Q35V3HBtvAZyo6xXWnM5VM2Kl5LvN%2FyGaglS3SA5RHsAAV%2FuZKuIV4715YnvvOgpxoP21AmHjLkTlcVGnalPJ6A4bFSuOHH1AoDt1XwophdYlI5pGTLuI7%2Fujz%2BhIOxhTX00CZtthObUmQ0QivqxKZrgarIQocEWVCm32%2FwUTeg1TD7QxKc%2FgV5TO7iXCpXtwpByOsHNByI3xCfuzLumSPkutWs6AOi0JxypzOvDXTMGlJpMWeqJNg%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/94466346/557270874/e90691de0f334b62cf2d8f7f82eceb55?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=iR4Tf%2B1vZGLNCdRl1e3cfjqlxI%2B1dcNzWEshrYfBKXnmVYhhEfWoPLsdoLxCFjhHdLM8cTsm%2BCxyGydq%2BEhbIokMyk7JxGY3Q35V3HBtvAZyo6xXWnM5VM2Kl5LvN%2FyGaglS3SA5RHsAAV%2FuZKuIV4715YnvvOgpxoP21AmHjLkTlcVGnalPJ6A4bFSuOHH1AoDt1XwophdYlI5pGTLuI7%2Fujz%2BhIOxhTX00CZtthObUmQ0QivqxKZrgarIQocEWVCm32%2FwUTeg1TD7QxKc%2FgV5TO7iXCpXtwpByOsHNByI3xCfuzLumSPkutWs6AOi0JxypzOvDXTMGlJpMWeqJNg%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 08:23:40,166 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': '6QaR3g8zS2LPLY9/guzrVQ==', 'Content-Type': 'application/json', 'Content-Length': '13956'}
2023-09-02 08:23:40,166 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 08:26:22,326 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/ucl_2023_com00087/trlx/an06fkb2/wandb-metadata.json?Expires=1693697418&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IyJ78E4KYvK780tDzzhndQFjwFadOPGOOCxnnEhzBCNSc7E8DaWyD7Lp%2FjsL4oWwssBH%2FxAkxObYd9Ui3BBe1qGBk3FBeebmV%2BvnaQayanzemeyJJ%2BmoiQ%2BGrkoM37rmz05qOw%2BVSOV9tAi1btjqK0w%2BibuxM3fIy1LvGk7wKOt61kqykMvq15jBIZogsrTY4aqjRbNU2Ynxr0dFwLwNagX07gZIpvze5lDOvj0bxGCSIf9cUqZg4rHwuS5UCmugAbtjVuoHbcXtPoZscv4XGd0EvURQ6kDGAfSgZnLVvEf6ITTyxaRoY1V427sqctkNNzOP%2Fn9xlgSqxJFQrhtVuA%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/ucl_2023_com00087/trlx/an06fkb2/wandb-metadata.json?Expires=1693697418&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IyJ78E4KYvK780tDzzhndQFjwFadOPGOOCxnnEhzBCNSc7E8DaWyD7Lp%2FjsL4oWwssBH%2FxAkxObYd9Ui3BBe1qGBk3FBeebmV%2BvnaQayanzemeyJJ%2BmoiQ%2BGrkoM37rmz05qOw%2BVSOV9tAi1btjqK0w%2BibuxM3fIy1LvGk7wKOt61kqykMvq15jBIZogsrTY4aqjRbNU2Ynxr0dFwLwNagX07gZIpvze5lDOvj0bxGCSIf9cUqZg4rHwuS5UCmugAbtjVuoHbcXtPoZscv4XGd0EvURQ6kDGAfSgZnLVvEf6ITTyxaRoY1V427sqctkNNzOP%2Fn9xlgSqxJFQrhtVuA%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 08:26:22,327 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '9452'}
2023-09-02 08:26:22,327 ERROR   wandb-upload_0:9010 [internal_api.py:upload_file():2103] upload_file response body: 
2023-09-02 08:29:42,105 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2099] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/94466346/557270874/e90691de0f334b62cf2d8f7f82eceb55?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=iR4Tf%2B1vZGLNCdRl1e3cfjqlxI%2B1dcNzWEshrYfBKXnmVYhhEfWoPLsdoLxCFjhHdLM8cTsm%2BCxyGydq%2BEhbIokMyk7JxGY3Q35V3HBtvAZyo6xXWnM5VM2Kl5LvN%2FyGaglS3SA5RHsAAV%2FuZKuIV4715YnvvOgpxoP21AmHjLkTlcVGnalPJ6A4bFSuOHH1AoDt1XwophdYlI5pGTLuI7%2Fujz%2BhIOxhTX00CZtthObUmQ0QivqxKZrgarIQocEWVCm32%2FwUTeg1TD7QxKc%2FgV5TO7iXCpXtwpByOsHNByI3xCfuzLumSPkutWs6AOi0JxypzOvDXTMGlJpMWeqJNg%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-artifacts-prod/wandb_artifacts/94466346/557270874/e90691de0f334b62cf2d8f7f82eceb55?Expires=1693698085&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=iR4Tf%2B1vZGLNCdRl1e3cfjqlxI%2B1dcNzWEshrYfBKXnmVYhhEfWoPLsdoLxCFjhHdLM8cTsm%2BCxyGydq%2BEhbIokMyk7JxGY3Q35V3HBtvAZyo6xXWnM5VM2Kl5LvN%2FyGaglS3SA5RHsAAV%2FuZKuIV4715YnvvOgpxoP21AmHjLkTlcVGnalPJ6A4bFSuOHH1AoDt1XwophdYlI5pGTLuI7%2Fujz%2BhIOxhTX00CZtthObUmQ0QivqxKZrgarIQocEWVCm32%2FwUTeg1TD7QxKc%2FgV5TO7iXCpXtwpByOsHNByI3xCfuzLumSPkutWs6AOi0JxypzOvDXTMGlJpMWeqJNg%3D%3D (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))
2023-09-02 08:29:42,105 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2101] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': '6QaR3g8zS2LPLY9/guzrVQ==', 'Content-Type': 'application/json', 'Content-Length': '13956'}
2023-09-02 08:29:42,105 ERROR   wandb-upload_2:9010 [internal_api.py:upload_file():2103] upload_file response body: 
